From 5ea7d31c9e0bacaec5129a997298448a641570f0 Mon Sep 17 00:00:00 2001
From: Tzung-Bi Shih <tzungbi@chromium.org>
Date: Tue, 20 Dec 2022 14:55:36 +0800
Subject: [PATCH] CHROMIUM: config: add initial skeleton

This is a squash of the following commits:

Part of "CHROMIUM: config: add configuration files and infrastructure"
CHROMIUM: config: update configs to latest in 5.15
CHROMIUM: kernel-rebase: normalization [autogenerated]
CHROMIUM: config: Enable PANEL_EDP where PANEL_SIMPLE was enabled
CHROMIUM: config: Enable CONFIG_DRM_PARADE_PS8640 for qcom
CHROMIUM: config: Enable DRM_PANEL_SAMSUNG_ATNA33XC20 for qcom
CHROMIUM: config: normalize configs
CHROMIUM: config: mediatek: enable panfrost driver
CHROMIUM: config: qcom: Enable qcom eDP phy driver
CHROMIUM: config: intel pineview: Fix default CPU freq governor
CHROMIUM: config: Enable ath10k debugging for rockchip64
CHROMIUM: config: Enable kcore on qcom platforms
CHROMIUM: config: rockchip: Drop Midgard configs
CHROMIUM: config: mediatek: Enable Mali Bifrost
CHROMIUM: config: mediatek: Add CONFIG_SX9324=m
CHROMIUM: config/pineview: Enable CHROMEOS_PRIVACY_SCREEN
CHROMIUM: config: enable CONFIG_ONBOARD_USB_HUB for QCA based devices
CHROMIUM: config: enable CONFIG_BPF_UNPRIV_DEFAULT_OFF
CHROMIUM: config: enable kernel module compression and decompression
CHROMIUM: config: enable CONFIG_XFRM_INTERFACE
CHROMIUM: config: Qualcomm and mediatek want to be PCIE supersavers
CHROMIUM: config: mediatek: remove CONFIG_ARM_MT8183_CCI_DEVFREQ
CHROMIUM: chromeos: enable CONFIG_BT_AOSPEXT in base.config
CHROMIUM: config: Enable platform code
CHROMIUM: config: Enable sysrq-x at early boot
CHROMIUM: config: Enable CONFIG_DEBUG_LIST.
intel-pineview/config: Enable X2APIC and IRQ_REMAP for MTL
CHROMIUM: config: Add NVME for qualcomm
CHROMIUM: config: Introduce config family for kernel build
CHROMIUM: config: Enable uid stats for qualcomm
CHROMIUM: config: Enable DRM_PANEL_BOE_TV101WUM_NL6 for Mrbland
CHROMIUM: config: Enable qcom SC7280 adsp bypass audio drivers
CHROMIUM: config: enable SX9324 SAR sensor for Qualcomm based boards
CHROMIUM: config: intel pineview: Enable MT7921E and BT_HCIBTUSB_MTK.
CHROMIUM: config: Built-in PowerVR driver 1.13
CHROMIUM: config: enable Landlock LSM.
CHROMIUM: config: enable dummy_hcd and mass_storage
CHROMIUM: config: intel pineview: Enable PCI bus based UFS driver.
CHROMIUM: config: intel-pineview: Enable UFS BSG node for provisioning.
CHROMIUM: config: update kernel configs via 'olddefconfig'
CHROMIUM: config: enable ftrace snapshots
CHROMIUM: Enable CONFIG_MWIFIEX_SDIO for mt8173 devices
CHROMIUM: defconfig: Support iwl7000 driver.
CHROMIUM: config: enable configs for zram writeback
CHROMIUM: config: Enable xt_qtaguid
CHROMIUM: config: amd stoneyride: Enable MT7921E and BT_HCIBTUSB_MTK.
CHROMIUM: config: intel-pineview: Enable SX9324
CHROMIUM: config: update kernel configs via 'olddefconfig'
CHROMIUM: config: enable additional tracing features
CHROMIUM: config: Enable lpass core clk controller on sc7280
CHROMIUM: Enable mediatek vcodec configs
CHROMIUM: config: Enable lpass pinctrl on sc7280
CHROMIUM: config: Enable hid-nintendo for Switch ProCon
CHROMIUM: config: Fix ipu3 (camera) support on soraka
CHROMIUM: config: Enable RCU callback offloading on all CPUs
CHROMIUM: config: Turn on Qualcomm LPG (PWM on the pmic)
CHROMIUM: defconfig: Added CONFIG_PINCTRL_ALDERLAKE=y for enabling ADL-N GPIO changes.
CHROMIUM: Enable BT component Marvell SDIO for MTK8173
CHROMIUM: config: stoneyridge: Remove Audio configs which is not used
CHROMIUM: config: Turn CONFIG_I2C_HID_OF_ELAN on for Qulcomm
CHROMIUM: config: amd stoneyridge: sync up with v5.15.46 merge
CHROMIUM: config: mediatek: enable CROS_EC_RPMSG and SCP support
CHROMIUM: config: mediatek: enable mtk cci devfreq
CHROMIUM: config: mediatek: enable TYPEC
CHROMIUM: config: sync up with v5.15.49 kernel config change
CHROMIUM: config: mediatek: enable SVS support
CHROMIUM: config: enable VIDEO_MEDIATEK_VPU for mt8173
CHROMIUM: config: stoneyridge: ADD Audio configs to Support Rembrandt Sof platform
CHROMIUM: configs: chromeos-intel-pineview: enable PXP
CHROMIUM: config: intel-pineview: Add Intel IOMMU, VFIO, PCI IOV.
CHROMIUM: config: Enable kprobes in general
CHROMIUM: config: Turn off USB_PCI on Qualcomm
CHROMIUM: config: Turn off img graphics driver on Qualcomm
CHROMIUM: config: Turn on CEC-related configs for x86
CHROMIUM: config: Enable CrOS EC CEC for fizz
CHROMIUM: config: mediatek: Enable audio support for mt8186
CHROMIUM: Select CONFIG_DEBUG_INFO_DWARF5
CHROMIUM: config: Enable CONFIG_SECURITY_LOADPIN_VERITY for Qualcomm
CHROMIUM: defconfig: Added CONFIG_PINCTRL_METEORLAKE=y for enabling
CHROMIUM: config: Enable CONFIG_UNICODE
CHROMIUM: config: Add CONFIG_HID_GOOGLE_HAMMER to qualcomm flavour
CHROMIUM: Disable IBPB on AMD Chromebooks
CHROMIUM: config: reven: Copy some changes from v5.10
CHROMIUM: config: enable Realtek 8852CE driver for skyrim
CHROMIUM: config: enable arm_smc watchdog timer
CHROMIUM: config: reven: Switch to efivarfs
CHROMIUM: config: reven: make legacy boot quiet
CHROMIUM: config: reven: enable Marvell PHY driver
CHROMIUM: config: Enable video MediaTek MDP3 configs
CHROMIUM: config: mediatek: Enable SOF
CHROMIUM: config: reven: enable Maxlinear PHY driver
CHROMIUM: config: mediatek: Enable thermal driver
CHROMIUM: enable Aquantia atlantic Thunderbolt Ethernet
CHROMIUM: chromeos-intel-denverton: Add SPI_INTEL_PCI=m
CHROMIUM: config: enable ov13b10 camera sensor
CHROMIUM: config: add img-rogue 1.13 for arm64/mediatek
CHROMIUM: config: PowerVR driver 1.17 as module
CHROMIUM: ALSA: set CONFIG_SND_DUMMY=m for testing purpose
CHROMIUM: config: reven: use modules for DRM drivers
CHROMIUM: enable CONFIG_SECURITY_SELINUX_PERMISSIVE_DONTAUDIT
CHROMIUM: Enable iwl7000 stoneyridge config
CHROMIUM: config: arm64: Build it6505/anx7625 as module
CHROMIUM: chromeos-intel-pineview: Add INTEL_HFI_THERMAL=y
CHROMIUM: Use upstream i2c cr50 driver on qualcomm
CHROMIUM: Use upstream CR50 I2C driver
CHROMIUM: config: Enable exfat kernel module
CHROMIUM: config: enable multi-gen LRU
CHROMIUM: config: mediatek: Enable spmi and mt6315
CHROMIUM: config: amd stoneyridge: Enable SP5100 watchdog
CHROMIUM: config: reven: Add support for amd renoir
CHROMIUM: config: reven: enable jasperlake pinctrl driver
CHROMIUM: config: Built-in PowerVR driver 1.17 and 1.13 as module
CHROMIUM: config: sync up with v5.15.75 kernel config change
CHROMIUM: config: reven: enable igc module for Intel ethernet controller I225-LM/I225-V
CHROMIUM: config: Make coresight drivers modular on qcom
CHROMIUM: config: mediatek: Set CONFIG_TOUCHSCREEN_MELFAS_MIP4
CHROMIUM: Enable AMDPSP I2C designware semaphore
CHROMIUM: config: enable IO_URING
CHROMIUM: config: Drop PowerVR 1.13 driver
CHROMIUM: config: mediatek: set CONFIG_UID_SYS_STATS
CHROMIUM: config: Disable vgem
CHROMIUM: config: mediatek: Set CONFIG_MTK_SOC_THERMAL
kernel-rebase: normalization [autogenerated]
CHROMIUM: config: mediatek: enable MTK_SOC_THERMAL
CHROMIUM: Enable NETFILTER_XT_MATCH_OWNER
config: Handle NVMEM config rename
CHROMIUM: config: Disable RT_GROUP_SCHED

Signed-off-by: Tzung-Bi Shih <tzungbi@chromium.org>
---

Notes:
    009356571453b8caac5f0234658f11e30138bd46
    d30483313dd880b6ad46f6e7a34eb687f68f0d6d

 .../arm64/chromiumos-arm64.flavour.config     | 301 +++++++++++
 .../arm64/chromiumos-mediatek.flavour.config  | 128 +++++
 .../arm64/chromiumos-qualcomm.flavour.config  | 157 ++++++
 .../chromiumos-rockchip64.flavour.config      |  93 ++++
 chromeos/config/chromeos/arm64/common.config  | 127 +++++
 .../armel/chromiumos-arm.flavour.config       |  50 ++
 .../armel/chromiumos-rockchip.flavour.config  |  13 +
 chromeos/config/chromeos/armel/common.config  | 142 ++++++
 chromeos/config/chromeos/base.config          | 477 ++++++++++++++++++
 .../chromeos-amd-stoneyridge.flavour.config   | 103 ++++
 .../chromeos-intel-denverton.flavour.config   |  62 +++
 .../chromeos-intel-pineview.flavour.config    | 192 +++++++
 .../chromeos-x86_64-reven.flavour.config      | 274 ++++++++++
 .../x86_64/chromiumos-x86_64.flavour.config   | 113 +++++
 chromeos/config/chromeos/x86_64/common.config | 145 ++++++
 15 files changed, 2377 insertions(+)
 create mode 100644 chromeos/config/chromeos/arm64/chromiumos-arm64.flavour.config
 create mode 100644 chromeos/config/chromeos/arm64/chromiumos-mediatek.flavour.config
 create mode 100644 chromeos/config/chromeos/arm64/chromiumos-qualcomm.flavour.config
 create mode 100644 chromeos/config/chromeos/arm64/chromiumos-rockchip64.flavour.config
 create mode 100644 chromeos/config/chromeos/arm64/common.config
 create mode 100644 chromeos/config/chromeos/armel/chromiumos-arm.flavour.config
 create mode 100644 chromeos/config/chromeos/armel/chromiumos-rockchip.flavour.config
 create mode 100644 chromeos/config/chromeos/armel/common.config
 create mode 100644 chromeos/config/chromeos/base.config
 create mode 100644 chromeos/config/chromeos/x86_64/chromeos-amd-stoneyridge.flavour.config
 create mode 100644 chromeos/config/chromeos/x86_64/chromeos-intel-denverton.flavour.config
 create mode 100644 chromeos/config/chromeos/x86_64/chromeos-intel-pineview.flavour.config
 create mode 100644 chromeos/config/chromeos/x86_64/chromeos-x86_64-reven.flavour.config
 create mode 100644 chromeos/config/chromeos/x86_64/chromiumos-x86_64.flavour.config
 create mode 100644 chromeos/config/chromeos/x86_64/common.config

diff --git a/chromeos/config/chromeos/arm64/chromiumos-arm64.flavour.config b/chromeos/config/chromeos/arm64/chromiumos-arm64.flavour.config
new file mode 100644
index 000000000000..c637e053dc2b
--- /dev/null
+++ b/chromeos/config/chromeos/arm64/chromiumos-arm64.flavour.config
@@ -0,0 +1,301 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_AK8975=m
+CONFIG_ARCH_MEDIATEK=y
+CONFIG_ARCH_QCOM=y
+CONFIG_ARCH_ROCKCHIP=y
+CONFIG_ARM_MEDIATEK_CCI_DEVFREQ=y
+CONFIG_ARM_MEDIATEK_CPUFREQ=y
+CONFIG_ARM_QCOM_CPUFREQ_HW=y
+CONFIG_ARM_SMMU=y
+CONFIG_ATA=y
+CONFIG_ATH10K_PCI=m
+CONFIG_ATH10K_SNOC=m
+CONFIG_ATH11K=m
+CONFIG_ATH11K_AHB=m
+CONFIG_ATH11K_DEBUG=y
+CONFIG_ATH11K_DEBUGFS=y
+CONFIG_ATH11K_TRACING=y
+CONFIG_BACKLIGHT_GPIO=y
+CONFIG_BLK_DEV_NVME=y
+CONFIG_BT_HCIBTUSB_MTK=y
+CONFIG_BT_HCIUART_QCA=y
+CONFIG_BT_HCIUART_RTL=y
+CONFIG_BT_MRVL=m
+CONFIG_BT_MRVL_SDIO=m
+CONFIG_BT_MTKSDIO=m
+# CONFIG_COMMON_CLK_MT2712 is not set
+# CONFIG_COMMON_CLK_MT6797 is not set
+CONFIG_COMMON_CLK_PALMAS=y
+CONFIG_COMMON_CLK_QCOM=y
+CONFIG_COMMON_CLK_RK808=y
+CONFIG_CORESIGHT=y
+CONFIG_CORESIGHT_CPU_DEBUG=y
+CONFIG_CORESIGHT_CPU_DEBUG_DEFAULT_ON=y
+CONFIG_CORESIGHT_LINK_AND_SINK_TMC=y
+CONFIG_CORESIGHT_SOURCE_ETM4X=y
+CONFIG_CPUFREQ_DT=y
+CONFIG_CROS_EC_MKBP_PROXIMITY=m
+CONFIG_CROS_EC_RPMSG=m
+CONFIG_CRYPTO_DEV_VIRTIO=m
+CONFIG_CRYPTO_MICHAEL_MIC=y
+CONFIG_DEVFREQ_THERMAL=y
+CONFIG_DRM_ANALOGIX_ANX7625=m
+CONFIG_DRM_ANALOGIX_ANX78XX=y
+CONFIG_DRM_ITE_IT6505=m
+CONFIG_DRM_MEDIATEK=y
+CONFIG_DRM_MEDIATEK_HDMI=y
+CONFIG_DRM_MSM=y
+# CONFIG_DRM_MSM_HDMI_HDCP is not set
+CONFIG_DRM_PANEL_BOE_TV101WUM_NL6=y
+CONFIG_DRM_PANEL_INNOLUX_HIMAX8279D=y
+CONFIG_DRM_PANEL_INNOLUX_P079ZCA=y
+CONFIG_DRM_PANEL_KINGDISPLAY_KD097D04=y
+CONFIG_DRM_PANEL_SAMSUNG_ATNA33XC20=y
+CONFIG_DRM_PANEL_VISIONOX_RM69299=y
+CONFIG_DRM_PANFROST=y
+CONFIG_DRM_PARADE_PS8640=y
+CONFIG_DRM_TI_SN65DSI86=y
+CONFIG_DRM_VIRTIO_GPU=y
+# CONFIG_DWMAC_IPQ806X is not set
+CONFIG_DW_WATCHDOG=y
+CONFIG_EDAC=y
+# CONFIG_EDAC_LEGACY_SYSFS is not set
+CONFIG_EDAC_QCOM=y
+CONFIG_EEPROM_AT24=y
+# CONFIG_EFIVAR_FS is not set
+CONFIG_EFI_GENERIC_STUB_INITRD_CMDLINE_LOADER=y
+CONFIG_EXTCON_PALMAS=y
+CONFIG_GENERIC_ADC_THERMAL=y
+CONFIG_GPIO_PALMAS=y
+CONFIG_HID_GOOGLE_HAMMER=m
+CONFIG_HWSPINLOCK=y
+CONFIG_HWSPINLOCK_QCOM=y
+CONFIG_HW_RANDOM_VIRTIO=y
+CONFIG_I2C_HID_OF=m
+CONFIG_I2C_HID_OF_ELAN=y
+CONFIG_I2C_HID_OF_GOODIX=y
+CONFIG_I2C_MT65XX=y
+CONFIG_I2C_QCOM_GENI=y
+CONFIG_I2C_QUP=y
+CONFIG_I2C_RK3X=y
+CONFIG_IIO_CROS_EC_ACCEL_LEGACY=m
+CONFIG_IIO_CROS_EC_SENSORS_LID_ANGLE=m
+CONFIG_INTERCONNECT=y
+CONFIG_INTERCONNECT_QCOM=y
+CONFIG_INTERCONNECT_QCOM_OSM_L3=y
+CONFIG_INTERCONNECT_QCOM_SC7180=y
+CONFIG_INTERCONNECT_QCOM_SC7280=y
+CONFIG_INTERCONNECT_QCOM_SDM845=y
+CONFIG_LEDS_CLASS_MULTICOLOR=y
+CONFIG_LEDS_PWM=y
+CONFIG_LEDS_QCOM_LPG=y
+CONFIG_MEDIATEK_MT6577_AUXADC=y
+CONFIG_MFD_MT6397=y
+CONFIG_MFD_PALMAS=y
+CONFIG_MFD_RK808=y
+CONFIG_MFD_SPMI_PMIC=y
+CONFIG_MMC_DW=y
+CONFIG_MMC_DW_ROCKCHIP=y
+CONFIG_MMC_MTK=y
+CONFIG_MMC_SDHCI_MSM=y
+CONFIG_MMC_SDHCI_OF_ARASAN=y
+CONFIG_MSM_GCC_8916=y
+CONFIG_MT7921E=m
+CONFIG_MT7921S=m
+CONFIG_MTK_CMDQ=y
+CONFIG_MTK_IOMMU=y
+CONFIG_MTK_PMIC_WRAP=y
+CONFIG_MTK_SENINF=m
+CONFIG_MTK_SOC_THERMAL=y
+CONFIG_MTK_SVS=y
+CONFIG_MTK_THERMAL=y
+CONFIG_MWIFIEX_PCIE=m
+CONFIG_MWIFIEX_SDIO=m
+CONFIG_PCIEAER=y
+CONFIG_PCIEASPM_POWER_SUPERSAVE=y
+CONFIG_PCIE_MEDIATEK=y
+CONFIG_PCIE_QCOM=y
+CONFIG_PCIE_ROCKCHIP_HOST=y
+CONFIG_PHY_MTK_TPHY=y
+CONFIG_PHY_QCOM_EDP=y
+CONFIG_PHY_QCOM_QMP=y
+CONFIG_PHY_QCOM_QUSB2=y
+CONFIG_PHY_QCOM_USB_SNPS_FEMTO_V2=y
+CONFIG_PHY_ROCKCHIP_DP=y
+CONFIG_PHY_ROCKCHIP_DPHY_RX0=m
+CONFIG_PHY_ROCKCHIP_EMMC=y
+CONFIG_PHY_ROCKCHIP_INNO_USB2=y
+CONFIG_PHY_ROCKCHIP_PCIE=y
+CONFIG_PHY_ROCKCHIP_TYPEC=y
+CONFIG_PHY_ROCKCHIP_USB=y
+CONFIG_PINCTRL_LPASS_LPI=y
+CONFIG_PINCTRL_MSM=y
+CONFIG_PINCTRL_MSM8916=y
+CONFIG_PINCTRL_QCOM_SPMI_PMIC=y
+CONFIG_PINCTRL_SC7180=y
+CONFIG_PINCTRL_SC7280=y
+CONFIG_PINCTRL_SC7280_LPASS_LPI=y
+CONFIG_PINCTRL_SDM845=y
+CONFIG_PL330_DMA=y
+CONFIG_PROC_KCORE=y
+CONFIG_PWM_MTK_DISP=y
+CONFIG_PWM_ROCKCHIP=y
+CONFIG_QCOM_A53PLL=y
+CONFIG_QCOM_AOSS_QMP=y
+CONFIG_QCOM_APCS_IPC=y
+CONFIG_QCOM_CLK_APCS_MSM8916=y
+CONFIG_QCOM_CLK_RPMH=y
+CONFIG_QCOM_COMMAND_DB=y
+# CONFIG_QCOM_EBI2 is not set
+CONFIG_QCOM_GENI_SE=y
+CONFIG_QCOM_IPA=m
+CONFIG_QCOM_IPCC=y
+CONFIG_QCOM_LLCC=y
+CONFIG_QCOM_PDC=y
+CONFIG_QCOM_Q6V5_ADSP=y
+CONFIG_QCOM_Q6V5_MSS=m
+CONFIG_QCOM_RMTFS_MEM=m
+CONFIG_QCOM_RPMH=y
+CONFIG_QCOM_RPMHPD=y
+CONFIG_QCOM_SMD_RPM=y
+CONFIG_QCOM_SMEM=y
+CONFIG_QCOM_SMP2P=y
+CONFIG_QCOM_SMSM=y
+CONFIG_QCOM_SOCINFO=y
+CONFIG_QCOM_SPMI_ADC5=m
+CONFIG_QCOM_SPMI_ADC_TM5=m
+CONFIG_QCOM_SPMI_TEMP_ALARM=m
+CONFIG_QCOM_STATS=m
+CONFIG_QCOM_SYSMON=y
+CONFIG_QCOM_WDT=y
+CONFIG_QRTR=y
+CONFIG_QRTR_SMD=y
+CONFIG_REGULATOR_CROS_EC=y
+CONFIG_REGULATOR_DA9211=y
+CONFIG_REGULATOR_MAX8973=y
+CONFIG_REGULATOR_MT6311=y
+CONFIG_REGULATOR_MT6358=y
+CONFIG_REGULATOR_MT6397=y
+CONFIG_REGULATOR_PALMAS=y
+CONFIG_REGULATOR_QCOM_RPMH=y
+CONFIG_REGULATOR_RK808=y
+CONFIG_REGULATOR_TPS51632=y
+CONFIG_REGULATOR_TPS62360=y
+CONFIG_REGULATOR_VCTRL=y
+CONFIG_REMOTEPROC=y
+CONFIG_RESET_QCOM_AOSS=y
+CONFIG_RESET_QCOM_PDC=y
+CONFIG_RESET_TI_SYSCON=y
+CONFIG_RMNET=y
+CONFIG_ROCKCHIP_IODOMAIN=y
+CONFIG_ROCKCHIP_PM_DOMAINS=y
+CONFIG_ROCKCHIP_SARADC=m
+CONFIG_ROCKCHIP_THERMAL=y
+CONFIG_RPMSG_CHAR=y
+CONFIG_RPMSG_QCOM_GLINK_SMEM=y
+CONFIG_RPMSG_QCOM_SMD=y
+CONFIG_RTC_DRV_MT6397=y
+CONFIG_RTC_DRV_PALMAS=y
+CONFIG_RTC_DRV_RK808=y
+CONFIG_RTW88=m
+CONFIG_RTW88_8822BE=m
+CONFIG_RTW88_8822CE=m
+CONFIG_RTW88_DEBUG=y
+CONFIG_RTW88_DEBUGFS=y
+CONFIG_SCSI_UFSHCD=y
+CONFIG_SCSI_UFSHCD_PLATFORM=y
+CONFIG_SCSI_UFS_QCOM=y
+CONFIG_SC_CAMCC_7180=y
+CONFIG_SC_CAMCC_7280=y
+CONFIG_SC_DISPCC_7180=y
+CONFIG_SC_DISPCC_7280=y
+CONFIG_SC_GPUCC_7180=y
+CONFIG_SC_GPUCC_7280=y
+CONFIG_SC_LPASSCC_7280=y
+CONFIG_SC_LPASS_CORECC_7180=y
+CONFIG_SC_LPASS_CORECC_7280=y
+CONFIG_SC_MSS_7180=m
+CONFIG_SC_VIDEOCC_7180=y
+CONFIG_SC_VIDEOCC_7280=y
+CONFIG_SDM_DISPCC_845=y
+CONFIG_SDM_GPUCC_845=y
+CONFIG_SDM_LPASSCC_845=y
+CONFIG_SDM_VIDEOCC_845=y
+CONFIG_SECURITY_LOADPIN_VERITY=y
+CONFIG_SENSORS_LM90=y
+CONFIG_SENSORS_TMP401=y
+CONFIG_SERIAL_8250=y
+CONFIG_SERIAL_8250_CONSOLE=y
+CONFIG_SERIAL_8250_DW=y
+CONFIG_SERIAL_8250_MT6577=y
+CONFIG_SERIAL_AMBA_PL011=y
+CONFIG_SERIAL_AMBA_PL011_CONSOLE=y
+CONFIG_SERIAL_DEV_BUS=y
+CONFIG_SERIAL_OF_PLATFORM=y
+CONFIG_SERIAL_QCOM_GENI=y
+CONFIG_SERIAL_QCOM_GENI_CONSOLE=y
+CONFIG_SKGE=m
+CONFIG_SKY2=m
+CONFIG_SND_DYNAMIC_MINORS=y
+CONFIG_SND_SOC_LPASS_VA_MACRO=y
+CONFIG_SND_SOC_MT8173=y
+CONFIG_SND_SOC_MT8173_RT5650=y
+CONFIG_SND_SOC_MT8173_RT5650_RT5514=y
+CONFIG_SND_SOC_MT8173_RT5650_RT5676=y
+CONFIG_SND_SOC_MT8183=y
+CONFIG_SND_SOC_MT8183_DA7219_MAX98357A=y
+CONFIG_SND_SOC_MT8183_MT6358_TS3A227E_MAX98357A=y
+CONFIG_SND_SOC_MT8186=y
+CONFIG_SND_SOC_MT8186_MT6366_DA7219_MAX98357=y
+CONFIG_SND_SOC_MT8192=y
+CONFIG_SND_SOC_MT8192_MT6359_RT1015_RT5682=y
+CONFIG_SND_SOC_QCOM=y
+CONFIG_SND_SOC_RK3399_GRU_SOUND=y
+CONFIG_SND_SOC_ROCKCHIP=y
+CONFIG_SND_SOC_ROCKCHIP_RT5645=y
+CONFIG_SND_SOC_ROCKCHIP_SPDIF=y
+CONFIG_SND_SOC_SC7180=m
+CONFIG_SND_SOC_SC7280=y
+CONFIG_SND_SOC_SSM4567=m
+CONFIG_SOUNDWIRE=y
+CONFIG_SOUNDWIRE_QCOM=y
+CONFIG_SPI_GPIO=y
+CONFIG_SPI_MT65XX=y
+CONFIG_SPI_MTK_NOR=y
+CONFIG_SPI_QCOM_GENI=y
+CONFIG_SPI_QCOM_QSPI=y
+CONFIG_SPI_ROCKCHIP=y
+CONFIG_SPMI=y
+CONFIG_STAGING_MEDIA=y
+CONFIG_STMMAC_ETH=m
+CONFIG_SX9310=m
+CONFIG_SX9324=m
+CONFIG_TCG_TIS_I2C_CR50=y
+CONFIG_TCG_TIS_I2C_INFINEON=y
+CONFIG_TOUCHSCREEN_MELFAS_MIP4=y
+CONFIG_TYPEC=m
+CONFIG_UID_SYS_STATS=y
+CONFIG_USB_MTU3=y
+CONFIG_USB_MTU3_HOST=y
+CONFIG_USB_ONBOARD_HUB=y
+CONFIG_USB_ULPI=y
+CONFIG_VIDEO_DW9768=m
+CONFIG_VIDEO_HANTRO=y
+CONFIG_VIDEO_MEDIATEK_JPEG=m
+CONFIG_VIDEO_MEDIATEK_MDP=m
+CONFIG_VIDEO_MEDIATEK_MDP3=m
+CONFIG_VIDEO_MEDIATEK_VCODEC=m
+CONFIG_VIDEO_MEDIATEK_VPU=y
+CONFIG_VIDEO_OV02A10=m
+CONFIG_VIDEO_OV2685=m
+CONFIG_VIDEO_OV5695=m
+CONFIG_VIDEO_OV8856=m
+CONFIG_VIDEO_ROCKCHIP_ISP1=m
+CONFIG_VIDEO_ROCKCHIP_VDEC=y
+CONFIG_VIRTIO_BLK=y
+CONFIG_VIRTIO_CONSOLE=y
+CONFIG_VIRTIO_INPUT=y
+CONFIG_VIRTIO_MMIO=y
+CONFIG_VIRTIO_NET=y
diff --git a/chromeos/config/chromeos/arm64/chromiumos-mediatek.flavour.config b/chromeos/config/chromeos/arm64/chromiumos-mediatek.flavour.config
new file mode 100644
index 000000000000..fc1df15e2aa2
--- /dev/null
+++ b/chromeos/config/chromeos/arm64/chromiumos-mediatek.flavour.config
@@ -0,0 +1,128 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ARCH_MEDIATEK=y
+CONFIG_ARM_MEDIATEK_CCI_DEVFREQ=y
+CONFIG_ARM_MEDIATEK_CPUFREQ=y
+CONFIG_ARM_SMC_WATCHDOG=y
+CONFIG_ATH10K_SDIO=m
+CONFIG_ATH10K_TRACING=y
+CONFIG_BT_HCIBTUSB_MTK=y
+CONFIG_BT_HCIUART_QCA=y
+CONFIG_BT_HCIUART_RTL=y
+CONFIG_BT_MRVL=m
+CONFIG_BT_MRVL_SDIO=m
+CONFIG_BT_MTKSDIO=m
+# CONFIG_COMMON_CLK_MT2712 is not set
+# CONFIG_COMMON_CLK_MT6797 is not set
+CONFIG_CONFIGFS_FS=y
+CONFIG_CROS_EC_RPMSG=m
+CONFIG_DRM_ANALOGIX_ANX7625=y
+CONFIG_DRM_ANALOGIX_ANX78XX=y
+CONFIG_DRM_ITE_IT6505=y
+CONFIG_DRM_MEDIATEK=y
+CONFIG_DRM_MEDIATEK_HDMI=y
+CONFIG_DRM_PANEL_BOE_TV101WUM_NL6=y
+CONFIG_DRM_PANEL_INNOLUX_HIMAX8279D=y
+CONFIG_DRM_PANEL_INNOLUX_P079ZCA=y
+CONFIG_DRM_PANFROST=y
+CONFIG_DRM_PARADE_PS8640=y
+CONFIG_DRM_POWERVR_ROGUE_1_17=y
+CONFIG_EEPROM_AT24=y
+# CONFIG_EFI is not set
+CONFIG_GENERIC_ADC_THERMAL=y
+CONFIG_HID_GOOGLE_HAMMER=m
+CONFIG_I2C_HID_OF=y
+CONFIG_I2C_HID_OF_GOODIX=y
+CONFIG_I2C_MT65XX=y
+CONFIG_IIO_CROS_EC_SENSORS_LID_ANGLE=m
+CONFIG_MALI_BIFROST=y
+CONFIG_MALI_BIFROST_EXPERT=y
+CONFIG_MALI_BIFROST_PLATFORM_NAME="mediatek"
+CONFIG_MEDIATEK_MT6577_AUXADC=y
+CONFIG_MFD_MT6397=y
+CONFIG_MMC_MTK=y
+CONFIG_MT7921E=m
+CONFIG_MT7921S=m
+CONFIG_MTK_ADSP_IPC=y
+CONFIG_MTK_ADSP_MBOX=y
+CONFIG_MTK_CMDQ=y
+CONFIG_MTK_IOMMU=y
+CONFIG_MTK_PMIC_WRAP=y
+CONFIG_MTK_SOC_THERMAL=y
+CONFIG_MTK_SOC_THERMAL_LVTS=y
+CONFIG_MTK_THERMAL=y
+CONFIG_MWIFIEX_SDIO=m
+CONFIG_MTK_SENINF=m
+CONFIG_MTK_SOC_THERMAL=y
+CONFIG_MTK_SVS=y
+CONFIG_MTK_THERMAL=y
+CONFIG_PCIEAER=y
+CONFIG_PCIEASPM_POWER_SUPERSAVE=y
+CONFIG_PCIE_MEDIATEK=y
+CONFIG_PHY_MTK_TPHY=y
+# CONFIG_PINCTRL_MT2712 is not set
+# CONFIG_PINCTRL_MT7622 is not set
+CONFIG_PL330_DMA=y
+# CONFIG_PTP_1588_CLOCK is not set
+CONFIG_PWM_MTK_DISP=y
+CONFIG_REGULATOR_CROS_EC=y
+CONFIG_REGULATOR_DA9211=y
+CONFIG_REGULATOR_MT6315=y
+CONFIG_REGULATOR_MT6358=y
+CONFIG_REGULATOR_MT6397=y
+CONFIG_REMOTEPROC=y
+CONFIG_RESET_TI_SYSCON=y
+CONFIG_RTC_DRV_MT6397=y
+CONFIG_RTW88=m
+CONFIG_RTW88_8822BE=m
+CONFIG_RTW88_8822CE=m
+CONFIG_RTW88_DEBUG=y
+CONFIG_RTW88_DEBUGFS=y
+CONFIG_SCSI=y
+CONFIG_SENSORS_TMP401=y
+CONFIG_SERIAL_8250=y
+CONFIG_SERIAL_8250_CONSOLE=y
+CONFIG_SERIAL_8250_MT6577=y
+CONFIG_SERIAL_DEV_BUS=y
+CONFIG_SERIAL_OF_PLATFORM=y
+CONFIG_SND_SOC_MT8173=y
+CONFIG_SND_SOC_MT8173_RT5650=y
+CONFIG_SND_SOC_MT8173_RT5650_RT5514=y
+CONFIG_SND_SOC_MT8173_RT5650_RT5676=y
+CONFIG_SND_SOC_MT8183=y
+CONFIG_SND_SOC_MT8183_DA7219_MAX98357A=y
+CONFIG_SND_SOC_MT8183_MT6358_TS3A227E_MAX98357A=y
+CONFIG_SND_SOC_MT8186=y
+CONFIG_SND_SOC_MT8186_MT6366_DA7219_MAX98357=y
+CONFIG_SND_SOC_MT8186_MT6366_RT1019_RT5682S=y
+CONFIG_SND_SOC_MT8192=y
+CONFIG_SND_SOC_MT8192_MT6359_RT1015_RT5682=y
+CONFIG_SND_SOC_SOF_MT8186=m
+CONFIG_SND_SOC_SOF_MTK_TOPLEVEL=y
+CONFIG_SND_SOC_SOF_OF=y
+CONFIG_SND_SOC_SOF_TOPLEVEL=y
+CONFIG_SPI_GPIO=y
+CONFIG_SPI_MT65XX=y
+CONFIG_SPI_MTK_NOR=y
+CONFIG_SPMI=y
+CONFIG_SPMI_MTK_PMIF=y
+CONFIG_STAGING_MEDIA=y
+CONFIG_SX9324=m
+CONFIG_TCG_TIS_I2C_CR50=y
+CONFIG_TCG_TIS_I2C_INFINEON=y
+CONFIG_TMPFS=y
+CONFIG_TOUCHSCREEN_MELFAS_MIP4=y
+CONFIG_TYPEC=y
+CONFIG_UID_SYS_STATS=y
+CONFIG_USB_MTU3=y
+CONFIG_USB_MTU3_HOST=y
+CONFIG_VIDEO_DW9768=m
+CONFIG_VIDEO_MEDIATEK_JPEG=m
+CONFIG_VIDEO_MEDIATEK_MDP=m
+CONFIG_VIDEO_MEDIATEK_MDP3=m
+CONFIG_VIDEO_MEDIATEK_VCODEC=m
+CONFIG_VIDEO_MEDIATEK_VPU=y
+CONFIG_VIDEO_OV02A10=m
+CONFIG_VIDEO_OV5695=m
+CONFIG_VIDEO_OV8856=m
diff --git a/chromeos/config/chromeos/arm64/chromiumos-qualcomm.flavour.config b/chromeos/config/chromeos/arm64/chromiumos-qualcomm.flavour.config
new file mode 100644
index 000000000000..45e314c55099
--- /dev/null
+++ b/chromeos/config/chromeos/arm64/chromiumos-qualcomm.flavour.config
@@ -0,0 +1,157 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ARCH_QCOM=y
+# CONFIG_ARM64_ERRATUM_819472 is not set
+# CONFIG_ARM64_ERRATUM_824069 is not set
+# CONFIG_ARM64_ERRATUM_826319 is not set
+# CONFIG_ARM64_ERRATUM_827319 is not set
+# CONFIG_ARM64_ERRATUM_832075 is not set
+# CONFIG_ARM64_ERRATUM_843419 is not set
+CONFIG_ARM_QCOM_CPUFREQ_HW=y
+CONFIG_ARM_SMMU=y
+CONFIG_ATH10K_SNOC=m
+CONFIG_ATH11K=m
+CONFIG_ATH11K_AHB=m
+CONFIG_ATH11K_DEBUG=y
+CONFIG_ATH11K_DEBUGFS=y
+CONFIG_ATH11K_TRACING=y
+CONFIG_BACKLIGHT_GPIO=y
+CONFIG_BLK_DEV_NVME=y
+CONFIG_BT_HCIUART_QCA=y
+# CONFIG_CAVIUM_ERRATUM_22375 is not set
+# CONFIG_CAVIUM_ERRATUM_23154 is not set
+CONFIG_COMMON_CLK_QCOM=y
+CONFIG_CONFIGFS_FS=y
+CONFIG_CORESIGHT=m
+CONFIG_CORESIGHT_LINK_AND_SINK_TMC=m
+CONFIG_CORESIGHT_SOURCE_ETM4X=m
+CONFIG_CROS_EC_MKBP_PROXIMITY=m
+CONFIG_CRYPTO_MICHAEL_MIC=y
+CONFIG_DEVFREQ_THERMAL=y
+CONFIG_DRM_MSM=y
+# CONFIG_DRM_MSM_DSI_14NM_PHY is not set
+# CONFIG_DRM_MSM_DSI_20NM_PHY is not set
+# CONFIG_DRM_MSM_DSI_28NM_8960_PHY is not set
+# CONFIG_DRM_MSM_DSI_28NM_PHY is not set
+# CONFIG_DRM_MSM_HDMI_HDCP is not set
+CONFIG_DRM_PANEL_BOE_TV101WUM_NL6=y
+CONFIG_DRM_PANEL_SAMSUNG_ATNA33XC20=y
+CONFIG_DRM_PANEL_VISIONOX_RM69299=y
+CONFIG_DRM_PARADE_PS8640=y
+CONFIG_DRM_TI_SN65DSI86=y
+CONFIG_EDAC=y
+# CONFIG_EDAC_LEGACY_SYSFS is not set
+CONFIG_EDAC_QCOM=y
+# CONFIG_EFI is not set
+CONFIG_EXTCON=y
+CONFIG_HID_GOOGLE_HAMMER=m
+CONFIG_HWSPINLOCK=y
+CONFIG_HWSPINLOCK_QCOM=y
+CONFIG_I2C_HID_OF=y
+CONFIG_I2C_HID_OF_ELAN=y
+CONFIG_I2C_HID_OF_GOODIX=y
+CONFIG_I2C_QCOM_GENI=y
+CONFIG_IIO_CROS_EC_SENSORS_LID_ANGLE=m
+CONFIG_INTERCONNECT=y
+CONFIG_INTERCONNECT_QCOM=y
+CONFIG_INTERCONNECT_QCOM_OSM_L3=y
+CONFIG_INTERCONNECT_QCOM_SC7180=y
+CONFIG_INTERCONNECT_QCOM_SC7280=y
+CONFIG_LEDS_CLASS_MULTICOLOR=y
+CONFIG_LEDS_PWM=y
+CONFIG_LEDS_QCOM_LPG=y
+CONFIG_MAILBOX=y
+CONFIG_MEMORY=y
+CONFIG_MFD_SPMI_PMIC=y
+CONFIG_MMC_SDHCI_MSM=y
+CONFIG_NVMEM_QCOM_QFPROM=y
+CONFIG_PCIEASPM_POWER_SUPERSAVE=y
+CONFIG_PCIE_QCOM=y
+CONFIG_PHY_QCOM_EDP=y
+CONFIG_PHY_QCOM_QMP=y
+CONFIG_PHY_QCOM_QUSB2=y
+CONFIG_PHY_QCOM_USB_SNPS_FEMTO_V2=y
+CONFIG_PINCTRL_LPASS_LPI=y
+CONFIG_PINCTRL_MSM=y
+CONFIG_PINCTRL_QCOM_SPMI_PMIC=y
+CONFIG_PINCTRL_SC7180=y
+CONFIG_PINCTRL_SC7280=y
+CONFIG_PINCTRL_SC7280_LPASS_LPI=y
+CONFIG_PM_DEVFREQ_EVENT=y
+CONFIG_PROC_KCORE=y
+# CONFIG_PTP_1588_CLOCK is not set
+CONFIG_QCOM_AOSS_QMP=y
+CONFIG_QCOM_APCS_IPC=y
+CONFIG_QCOM_CLK_RPMH=y
+CONFIG_QCOM_COMMAND_DB=y
+# CONFIG_QCOM_EBI2 is not set
+CONFIG_QCOM_GENI_SE=y
+CONFIG_QCOM_IPA=y
+CONFIG_QCOM_IPCC=y
+CONFIG_QCOM_LLCC=y
+CONFIG_QCOM_PDC=y
+CONFIG_QCOM_Q6V5_ADSP=y
+CONFIG_QCOM_Q6V5_MSS=y
+CONFIG_QCOM_RMTFS_MEM=y
+CONFIG_QCOM_RPMH=y
+CONFIG_QCOM_RPMHPD=y
+CONFIG_QCOM_SMEM=y
+CONFIG_QCOM_SMP2P=y
+CONFIG_QCOM_SOCINFO=y
+CONFIG_QCOM_SPMI_ADC5=m
+CONFIG_QCOM_SPMI_ADC_TM5=m
+CONFIG_QCOM_SPMI_TEMP_ALARM=m
+CONFIG_QCOM_STATS=m
+CONFIG_QCOM_SYSMON=y
+CONFIG_QCOM_WDT=y
+CONFIG_QRTR=y
+CONFIG_QRTR_SMD=y
+CONFIG_RAS=y
+CONFIG_REGULATOR_QCOM_RPMH=y
+CONFIG_REMOTEPROC=y
+CONFIG_RESET_QCOM_AOSS=y
+CONFIG_RESET_QCOM_PDC=y
+CONFIG_RMNET=y
+CONFIG_RPMSG_CHAR=y
+CONFIG_RPMSG_QCOM_GLINK_SMEM=y
+CONFIG_RPMSG_QCOM_SMD=y
+CONFIG_SCSI=y
+CONFIG_SCSI_UFSHCD=y
+CONFIG_SCSI_UFSHCD_PLATFORM=y
+CONFIG_SCSI_UFS_QCOM=y
+CONFIG_SC_CAMCC_7180=y
+CONFIG_SC_CAMCC_7280=y
+CONFIG_SC_DISPCC_7180=y
+CONFIG_SC_DISPCC_7280=y
+CONFIG_SC_GPUCC_7180=y
+CONFIG_SC_GPUCC_7280=y
+CONFIG_SC_LPASSCC_7280=y
+CONFIG_SC_LPASS_CORECC_7180=y
+CONFIG_SC_LPASS_CORECC_7280=y
+CONFIG_SC_MSS_7180=m
+CONFIG_SC_VIDEOCC_7180=y
+CONFIG_SC_VIDEOCC_7280=y
+CONFIG_SECURITY_LOADPIN_VERITY=y
+CONFIG_SERIAL_DEV_BUS=y
+CONFIG_SERIAL_QCOM_GENI=y
+CONFIG_SERIAL_QCOM_GENI_CONSOLE=y
+CONFIG_SND_DYNAMIC_MINORS=y
+CONFIG_SND_SOC_LPASS_VA_MACRO=y
+CONFIG_SND_SOC_QCOM=y
+CONFIG_SND_SOC_SC7180=m
+CONFIG_SND_SOC_SC7280=y
+CONFIG_SOUNDWIRE=y
+CONFIG_SOUNDWIRE_QCOM=y
+CONFIG_SPI_QCOM_GENI=y
+CONFIG_SPI_QCOM_QSPI=y
+CONFIG_SPMI=y
+CONFIG_SX9310=m
+CONFIG_SX9324=m
+CONFIG_TCG_TIS_I2C_CR50=y
+CONFIG_TYPEC=m
+CONFIG_UID_SYS_STATS=y
+# CONFIG_USB_DWC3_OF_SIMPLE is not set
+CONFIG_USB_ONBOARD_HUB=y
+# CONFIG_USB_PCI is not set
+CONFIG_VIDEO_QCOM_VENUS=m
diff --git a/chromeos/config/chromeos/arm64/chromiumos-rockchip64.flavour.config b/chromeos/config/chromeos/arm64/chromiumos-rockchip64.flavour.config
new file mode 100644
index 000000000000..219f85a6dad3
--- /dev/null
+++ b/chromeos/config/chromeos/arm64/chromiumos-rockchip64.flavour.config
@@ -0,0 +1,93 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_AK8975=m
+CONFIG_ARCH_ROCKCHIP=y
+# CONFIG_ARM64_ERRATUM_819472 is not set
+# CONFIG_ARM64_ERRATUM_824069 is not set
+# CONFIG_ARM64_ERRATUM_826319 is not set
+# CONFIG_ARM64_ERRATUM_827319 is not set
+# CONFIG_ARM64_ERRATUM_832075 is not set
+# CONFIG_ARM64_ERRATUM_843419 is not set
+# CONFIG_ARM64_USE_LSE_ATOMICS is not set
+CONFIG_ARM_RK3399_DMC_DEVFREQ=y
+CONFIG_ATH10K_PCI=m
+CONFIG_BT_HCIUART_H4=y
+CONFIG_BT_MRVL=m
+CONFIG_BT_MRVL_SDIO=m
+# CONFIG_CAVIUM_ERRATUM_22375 is not set
+# CONFIG_CAVIUM_ERRATUM_23154 is not set
+CONFIG_COMMON_CLK_RK808=y
+CONFIG_CORESIGHT=y
+CONFIG_CORESIGHT_CPU_DEBUG=y
+CONFIG_CORESIGHT_CPU_DEBUG_DEFAULT_ON=y
+CONFIG_CPUFREQ_DT=y
+CONFIG_DEVFREQ_THERMAL=y
+CONFIG_DRM_PANEL_INNOLUX_P079ZCA=y
+CONFIG_DRM_PANEL_KINGDISPLAY_KD097D04=y
+CONFIG_DRM_ROCKCHIP=y
+CONFIG_DW_WATCHDOG=y
+# CONFIG_EFI is not set
+CONFIG_I2C_HID_OF=y
+CONFIG_I2C_RK3X=y
+CONFIG_IIO_CROS_EC_ACCEL_LEGACY=m
+CONFIG_MAILBOX=y
+CONFIG_MEMORY=y
+CONFIG_MFD_RK808=y
+CONFIG_MMC_DW=y
+CONFIG_MMC_DW_ROCKCHIP=y
+CONFIG_MMC_SDHCI_OF_ARASAN=y
+CONFIG_MWIFIEX_PCIE=m
+CONFIG_MWIFIEX_SDIO=m
+CONFIG_PCIEAER=y
+CONFIG_PCIEASPM_POWERSAVE=y
+CONFIG_PCIE_ROCKCHIP_HOST=y
+CONFIG_PHY_ROCKCHIP_DP=y
+CONFIG_PHY_ROCKCHIP_DPHY_RX0=m
+CONFIG_PHY_ROCKCHIP_EMMC=y
+CONFIG_PHY_ROCKCHIP_INNO_USB2=y
+CONFIG_PHY_ROCKCHIP_PCIE=y
+CONFIG_PHY_ROCKCHIP_TYPEC=y
+CONFIG_PHY_ROCKCHIP_USB=y
+CONFIG_PL330_DMA=y
+CONFIG_PM_DEVFREQ=y
+CONFIG_PTP_1588_CLOCK=m
+CONFIG_PWM_ROCKCHIP=y
+CONFIG_REGULATOR=y
+CONFIG_REGULATOR_RK808=y
+CONFIG_REGULATOR_VCTRL=y
+CONFIG_ROCKCHIP_ANALOGIX_DP=y
+CONFIG_ROCKCHIP_CDN_DP=y
+CONFIG_ROCKCHIP_DW_MIPI_DSI=y
+CONFIG_ROCKCHIP_IODOMAIN=y
+CONFIG_ROCKCHIP_IOMMU=y
+CONFIG_ROCKCHIP_PM_DOMAINS=y
+CONFIG_ROCKCHIP_SARADC=m
+CONFIG_ROCKCHIP_THERMAL=y
+CONFIG_RTC_DRV_RK808=y
+CONFIG_SCSI=y
+CONFIG_SENSORS_NTC_THERMISTOR=m
+CONFIG_SERIAL_8250=y
+CONFIG_SERIAL_8250_CONSOLE=y
+# CONFIG_SERIAL_8250_DEPRECATED_OPTIONS is not set
+CONFIG_SERIAL_8250_DW=y
+CONFIG_SERIAL_OF_PLATFORM=y
+CONFIG_SND_DYNAMIC_MINORS=y
+CONFIG_SND_SOC_RK3399_GRU_SOUND=y
+CONFIG_SND_SOC_ROCKCHIP=y
+CONFIG_SND_SOC_ROCKCHIP_RT5645=y
+CONFIG_SND_SOC_ROCKCHIP_SPDIF=y
+CONFIG_SND_SOC_SSM4567=m
+CONFIG_SND_SOC_TS3A227E=y
+CONFIG_SPI_BITBANG=m
+CONFIG_SPI_ROCKCHIP=y
+CONFIG_STAGING_MEDIA=y
+CONFIG_STMMAC_ETH=m
+CONFIG_TCG_TIS_I2C_INFINEON=y
+CONFIG_TMPFS=y
+CONFIG_USB_ULPI=y
+CONFIG_VIDEO_HANTRO=y
+CONFIG_VIDEO_OV2685=m
+CONFIG_VIDEO_OV5695=m
+CONFIG_VIDEO_ROCKCHIP_ISP1=m
+CONFIG_VIDEO_ROCKCHIP_VDEC=y
diff --git a/chromeos/config/chromeos/arm64/common.config b/chromeos/config/chromeos/arm64/common.config
new file mode 100644
index 000000000000..7dfae0b3de00
--- /dev/null
+++ b/chromeos/config/chromeos/arm64/common.config
@@ -0,0 +1,127 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ALT_SYSCALL=y
+CONFIG_ALT_SYSCALL_CHROMIUMOS=y
+CONFIG_ARCH_MMAP_RND_BITS=24
+CONFIG_ARCH_MMAP_RND_COMPAT_BITS=16
+# CONFIG_ARM64_BTI is not set
+CONFIG_ARM64_SW_TTBR0_PAN=y
+CONFIG_ARMV8_DEPRECATED=y
+CONFIG_ARM_PSCI_CPUIDLE=y
+CONFIG_ATH10K=m
+CONFIG_ATH10K_DEBUG=y
+CONFIG_ATH10K_DEBUGFS=y
+CONFIG_BACKLIGHT_CLASS_DEVICE=y
+CONFIG_BACKLIGHT_PWM=y
+CONFIG_BATTERY_SBS=y
+CONFIG_BT_HCIBTSDIO=m
+CONFIG_BT_HCIUART=m
+# CONFIG_CAVIUM_ERRATUM_27456 is not set
+CONFIG_CFG80211_WEXT=y
+CONFIG_CHARGER_CROS_USBPD=y
+CONFIG_CHARGER_GPIO=y
+CONFIG_COMPAT=y
+CONFIG_CP15_BARRIER_EMULATION=y
+CONFIG_CPU_BOOST=y
+CONFIG_CPU_FREQ=y
+CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE=y
+CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
+CONFIG_CPU_FREQ_GOV_ONDEMAND=y
+CONFIG_CPU_FREQ_GOV_POWERSAVE=y
+CONFIG_CPU_FREQ_GOV_SCHEDUTIL=y
+CONFIG_CPU_IDLE=y
+CONFIG_CPU_THERMAL=y
+# CONFIG_CROS_EC_LIGHTBAR is not set
+# CONFIG_CROS_EC_VBC is not set
+CONFIG_CRYPTO_AES_ARM64=y
+CONFIG_CRYPTO_AES_ARM64_CE_BLK=y
+CONFIG_CRYPTO_AES_ARM64_CE_CCM=y
+CONFIG_CRYPTO_AES_ARM64_NEON_BLK=y
+CONFIG_CRYPTO_CRYPTD=y
+CONFIG_CRYPTO_GHASH_ARM64_CE=y
+CONFIG_CRYPTO_LZO=m
+CONFIG_CRYPTO_SHA1_ARM64_CE=y
+CONFIG_CRYPTO_SHA2_ARM64_CE=y
+CONFIG_DEBUG_GPIO=y
+CONFIG_DEVFREQ_GOV_PERFORMANCE=y
+CONFIG_DEVFREQ_GOV_POWERSAVE=y
+CONFIG_DEVFREQ_GOV_USERSPACE=y
+CONFIG_DEVTMPFS_SAFE=y
+CONFIG_DMADEVICES=y
+CONFIG_DRM_PANEL_EDP=y
+CONFIG_DRM_PANEL_SIMPLE=y
+CONFIG_ENERGY_MODEL=y
+CONFIG_EXTCON_USBC_CROS_EC=y
+CONFIG_GPIO_SYSFS=y
+CONFIG_HARDLOCKUP_DETECTOR_BUDDY_CPU=y
+CONFIG_HID_RMI=m
+CONFIG_HZ_1000=y
+CONFIG_I2C_CHARDEV=y
+CONFIG_I2C_CROS_EC_TUNNEL=y
+CONFIG_IIO=y
+CONFIG_IIO_CROS_EC_ACTIVITY=m
+CONFIG_IIO_CROS_EC_BARO=m
+CONFIG_IIO_CROS_EC_LIGHT_PROX=m
+CONFIG_IIO_CROS_EC_SENSORS=m
+CONFIG_IIO_CROS_EC_SENSORS_SYNC=m
+CONFIG_IRQ_TIME_ACCOUNTING=y
+CONFIG_JUMP_LABEL=y
+# CONFIG_KEYBOARD_ATKBD is not set
+CONFIG_KEYBOARD_GPIO=y
+CONFIG_KFENCE=y
+CONFIG_KFENCE_SAMPLE_INTERVAL=500
+CONFIG_MAC80211_DEBUGFS=y
+CONFIG_MAC80211_LEDS=y
+CONFIG_MMC_SDHCI_PLTFM=y
+# CONFIG_MOUSE_PS2 is not set
+CONFIG_MTD=y
+CONFIG_MTD_CMDLINE_PARTS=y
+CONFIG_MTD_PARTITIONED_MASTER=y
+CONFIG_MTD_SPI_NOR=y
+# CONFIG_NET_VENDOR_VERTEXCOM is not set
+CONFIG_NR_CPUS=8
+CONFIG_PCI=y
+CONFIG_PCIEPORTBUS=y
+CONFIG_PHYLIB=y
+CONFIG_PSTORE=y
+CONFIG_PWM=y
+CONFIG_PWM_CROS_EC=y
+CONFIG_RANDOMIZE_BASE=y
+CONFIG_REGULATOR_FIXED_VOLTAGE=y
+CONFIG_REGULATOR_GPIO=y
+CONFIG_REGULATOR_PWM=y
+CONFIG_REGULATOR_USERSPACE_CONSUMER=y
+CONFIG_REGULATOR_VIRTUAL_CONSUMER=y
+CONFIG_RTC_DRV_CROS_EC=y
+CONFIG_SCHED_MC=y
+CONFIG_SECURITY_CHROMIUMOS=y
+CONFIG_SERIO=m
+CONFIG_SETEND_EMULATION=y
+CONFIG_SND_SIMPLE_CARD=y
+CONFIG_SWP_EMULATION=y
+CONFIG_TCG_TIS_SPI=y
+CONFIG_TCG_TIS_SPI_CR50=y
+CONFIG_THERMAL=y
+CONFIG_THERMAL_DEFAULT_GOV_POWER_ALLOCATOR=y
+CONFIG_THERMAL_GOV_POWER_ALLOCATOR=y
+CONFIG_THERMAL_GOV_STEP_WISE=y
+CONFIG_THERMAL_GOV_USER_SPACE=y
+CONFIG_THERMAL_WRITABLE_TRIPS=y
+CONFIG_UCLAMP_TASK=y
+CONFIG_UCLAMP_TASK_GROUP=y
+CONFIG_USB_DWC3=y
+CONFIG_USB_EHCI_HCD_PLATFORM=y
+CONFIG_USB_EHCI_ROOT_HUB_TT=y
+CONFIG_USB_GADGET=y
+CONFIG_USB_OHCI_HCD=y
+CONFIG_USB_OHCI_HCD_PLATFORM=y
+CONFIG_USB_XHCI_HCD=y
+CONFIG_V4L_MEM2MEM_DRIVERS=y
+CONFIG_V4L_PLATFORM_DRIVERS=y
+# CONFIG_XZ_DEC_ARM is not set
+# CONFIG_XZ_DEC_ARMTHUMB is not set
+# CONFIG_XZ_DEC_X86 is not set
+CONFIG_ZRAM=m
+CONFIG_ZRAM_MEMORY_TRACKING=y
+CONFIG_ZRAM_WRITEBACK=y
diff --git a/chromeos/config/chromeos/armel/chromiumos-arm.flavour.config b/chromeos/config/chromeos/armel/chromiumos-arm.flavour.config
new file mode 100644
index 000000000000..bffaa991e6bf
--- /dev/null
+++ b/chromeos/config/chromeos/armel/chromiumos-arm.flavour.config
@@ -0,0 +1,50 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ARCH_TEGRA=y
+CONFIG_ARCH_TEGRA_114_SOC=y
+CONFIG_ARCH_TEGRA_124_SOC=y
+CONFIG_ARCH_TEGRA_2x_SOC=y
+CONFIG_ARCH_TEGRA_3x_SOC=y
+CONFIG_ARM_LPAE=y
+# CONFIG_ARM_TEGRA124_CPUFREQ is not set
+CONFIG_ATH6KL=m
+CONFIG_ATH6KL_DEBUG=y
+CONFIG_B43=m
+CONFIG_B43_SDIO=y
+CONFIG_CHARGER_TPS65090=y
+CONFIG_GPIO_TPS6586X=y
+CONFIG_HID_RMI=m
+CONFIG_I2C_MUX_PINCTRL=y
+CONFIG_I2C_TEGRA=y
+CONFIG_KEYBOARD_NVEC=y
+CONFIG_KEYBOARD_TEGRA=y
+CONFIG_MAILBOX=y
+CONFIG_MFD_AS3722=y
+CONFIG_MFD_NVEC=y
+CONFIG_MFD_TPS65090=y
+CONFIG_MMC_SDHCI_TEGRA=y
+CONFIG_NVEC_PAZ00=y
+CONFIG_NVEC_POWER=y
+CONFIG_PINCTRL_AS3722=y
+CONFIG_POWER_RESET_AS3722=y
+CONFIG_PWM_TEGRA=y
+CONFIG_REGULATOR_AS3722=y
+CONFIG_REGULATOR_TPS51632=y
+CONFIG_REGULATOR_TPS62360=y
+CONFIG_REGULATOR_TPS65090=y
+CONFIG_RTC_DRV_AS3722=y
+CONFIG_RTC_DRV_TEGRA=y
+CONFIG_SENSORS_LM90=y
+CONFIG_SERIO=m
+CONFIG_SMC91X=m
+CONFIG_SND_HDA_CODEC_HDMI=y
+CONFIG_SND_HDA_GENERIC=y
+CONFIG_SND_HDA_TEGRA=y
+CONFIG_SPI_TEGRA20_SFLASH=y
+CONFIG_TEGRA_IOMMU_GART=y
+CONFIG_TEGRA_IOMMU_SMMU=y
+CONFIG_USB_DWC2_HOST=y
+CONFIG_USB_EHCI_TEGRA=y
+CONFIG_USB_OHCI_HCD=y
+CONFIG_USB_XHCI_HCD=y
diff --git a/chromeos/config/chromeos/armel/chromiumos-rockchip.flavour.config b/chromeos/config/chromeos/armel/chromiumos-rockchip.flavour.config
new file mode 100644
index 000000000000..6030d41c25cc
--- /dev/null
+++ b/chromeos/config/chromeos/armel/chromiumos-rockchip.flavour.config
@@ -0,0 +1,13 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_BCMA=m
+CONFIG_DEVFREQ_GOV_SIMPLE_ONDEMAND=y
+CONFIG_HID_RMI=y
+CONFIG_PM_DEVFREQ=y
+CONFIG_SND_DYNAMIC_MINORS=y
+CONFIG_SND_SOC_TLV320AIC23_I2C=y
+CONFIG_SND_SOC_WM8753=y
+CONFIG_SND_SOC_WM8903=y
+CONFIG_USB_EHCI_ROOT_HUB_TT=y
+CONFIG_USB_GADGET=m
diff --git a/chromeos/config/chromeos/armel/common.config b/chromeos/config/chromeos/armel/common.config
new file mode 100644
index 000000000000..3ebf395bfb63
--- /dev/null
+++ b/chromeos/config/chromeos/armel/common.config
@@ -0,0 +1,142 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ARCH_MMAP_RND_BITS=16
+CONFIG_ARCH_ROCKCHIP=y
+# CONFIG_ARM_ERRATA_643719 is not set
+CONFIG_ARM_ERRATA_818325_852422=y
+CONFIG_ARM_ERRATA_821420=y
+CONFIG_ARM_ERRATA_825619=y
+CONFIG_ARM_ERRATA_857271=y
+# CONFIG_ARM_MODULE_PLTS is not set
+CONFIG_ARM_THUMBEE=y
+CONFIG_BACKLIGHT_CLASS_DEVICE=y
+CONFIG_BACKLIGHT_PWM=y
+CONFIG_BATTERY_BQ27XXX=y
+CONFIG_BATTERY_SBS=y
+CONFIG_BRCMFMAC=m
+CONFIG_BT_HCIBTSDIO=m
+CONFIG_BT_HCIUART=m
+CONFIG_BT_HCIUART_ATH3K=y
+CONFIG_BT_MRVL=m
+CONFIG_BT_MRVL_SDIO=m
+CONFIG_CFG80211_WEXT=y
+CONFIG_CHARGER_BQ24735=y
+CONFIG_CHARGER_GPIO=y
+CONFIG_COMMON_CLK_RK808=y
+CONFIG_CONFIGFS_FS=y
+CONFIG_CPUFREQ_DT=y
+CONFIG_CPU_BOOST=y
+CONFIG_CPU_FREQ=y
+CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE=y
+CONFIG_CPU_FREQ_GOV_ONDEMAND=y
+CONFIG_CPU_FREQ_GOV_SCHEDUTIL=y
+CONFIG_CPU_IDLE=y
+CONFIG_CPU_THERMAL=y
+# CONFIG_CROS_EC_VBC is not set
+CONFIG_CRYPTO_AES_ARM=y
+CONFIG_CRYPTO_AES_ARM_BS=y
+CONFIG_CRYPTO_LZO=m
+CONFIG_CRYPTO_SHA1_ARM_NEON=y
+CONFIG_CRYPTO_SHA256_ARM=y
+CONFIG_CRYPTO_SHA512_ARM=y
+CONFIG_DEBUG_GPIO=y
+CONFIG_DEVFREQ_GOV_PERFORMANCE=y
+CONFIG_DEVFREQ_GOV_POWERSAVE=y
+CONFIG_DEVFREQ_GOV_USERSPACE=y
+CONFIG_DEVFREQ_THERMAL=y
+CONFIG_DEVTMPFS_SAFE=y
+CONFIG_DMADEVICES=y
+CONFIG_DRM_DW_HDMI_AHB_AUDIO=y
+CONFIG_DRM_DW_HDMI_I2S_AUDIO=y
+CONFIG_DRM_PANEL_EDP=y
+CONFIG_DRM_PANEL_SIMPLE=y
+CONFIG_DRM_ROCKCHIP=y
+CONFIG_DW_WATCHDOG=y
+CONFIG_GPIO_SYSFS=y
+CONFIG_HARDLOCKUP_DETECTOR_BUDDY_CPU=y
+CONFIG_HIGHMEM=y
+# CONFIG_HIGHPTE is not set
+CONFIG_HZ_250=y
+CONFIG_I2C_CHARDEV=m
+CONFIG_I2C_CROS_EC_TUNNEL=y
+CONFIG_I2C_HID_OF=m
+CONFIG_I2C_RK3X=y
+CONFIG_IIO=y
+CONFIG_IIO_CROS_EC_ACCEL_LEGACY=m
+CONFIG_KERNEL_MODE_NEON=y
+# CONFIG_KEYBOARD_ATKBD is not set
+CONFIG_KEYBOARD_GPIO=y
+CONFIG_KEYBOARD_GPIO_POLLED=y
+CONFIG_LEDS_GPIO=y
+CONFIG_LSM="landlock,lockdown,yama,loadpin,safesetid,integrity,selinux,smack,tomoyo,apparmor"
+CONFIG_MAC80211_DEBUGFS=y
+CONFIG_MAC80211_LEDS=y
+CONFIG_MEMORY=y
+CONFIG_MFD_RK808=y
+CONFIG_MFD_TPS6586X=y
+CONFIG_MMC_DW=y
+CONFIG_MMC_DW_ROCKCHIP=y
+CONFIG_MMC_SDHCI_PLTFM=y
+# CONFIG_MOUSE_PS2 is not set
+CONFIG_MTD=y
+CONFIG_MTD_CMDLINE_PARTS=y
+CONFIG_MTD_PARTITIONED_MASTER=y
+CONFIG_MTD_SPI_NOR=y
+CONFIG_MWIFIEX_SDIO=m
+CONFIG_NEON=y
+# CONFIG_NET_VENDOR_VERTEXCOM is not set
+CONFIG_PHYLIB=y
+CONFIG_PHY_ROCKCHIP_DP=y
+CONFIG_PHY_ROCKCHIP_USB=y
+CONFIG_PL330_DMA=y
+CONFIG_POWER_RESET=y
+CONFIG_POWER_RESET_GPIO_RESTART=y
+CONFIG_PPS=y
+CONFIG_PSTORE=y
+CONFIG_PTP_1588_CLOCK=m
+CONFIG_PWM=y
+CONFIG_PWM_ROCKCHIP=y
+CONFIG_REALTEK_PHY=y
+CONFIG_REGULATOR_FIXED_VOLTAGE=y
+CONFIG_REGULATOR_PWM=y
+CONFIG_REGULATOR_RK808=y
+CONFIG_REGULATOR_TPS6586X=y
+CONFIG_ROCKCHIP_ANALOGIX_DP=y
+CONFIG_ROCKCHIP_DW_HDMI=y
+CONFIG_ROCKCHIP_DW_MIPI_DSI=y
+CONFIG_ROCKCHIP_INNO_HDMI=y
+CONFIG_ROCKCHIP_IODOMAIN=y
+CONFIG_ROCKCHIP_IOMMU=y
+CONFIG_ROCKCHIP_PM_DOMAINS=y
+CONFIG_ROCKCHIP_SARADC=m
+CONFIG_ROCKCHIP_THERMAL=y
+CONFIG_RTC_DRV_RK808=y
+CONFIG_SCSI=y
+CONFIG_SERIAL_8250=y
+CONFIG_SERIAL_8250_CONSOLE=y
+CONFIG_SERIAL_8250_DW=y
+CONFIG_SERIAL_OF_PLATFORM=y
+CONFIG_SMP=y
+# CONFIG_SND_ARM is not set
+CONFIG_SND_SOC_ROCKCHIP=y
+CONFIG_SND_SOC_ROCKCHIP_MAX98090=y
+CONFIG_SPI_BITBANG=m
+CONFIG_SPI_ROCKCHIP=y
+CONFIG_STAGING_MEDIA=y
+CONFIG_STMMAC_ETH=m
+CONFIG_TCG_TIS_I2C_INFINEON=y
+CONFIG_THERMAL=y
+CONFIG_THERMAL_GOV_USER_SPACE=y
+CONFIG_TMPFS=y
+CONFIG_UCLAMP_TASK=y
+CONFIG_UCLAMP_TASK_GROUP=y
+CONFIG_USB_DWC2=y
+CONFIG_USB_EHCI_HCD_PLATFORM=y
+CONFIG_V4L_MEM2MEM_DRIVERS=y
+CONFIG_VFP=y
+CONFIG_VIDEO_HANTRO=y
+# CONFIG_XZ_DEC_X86 is not set
+CONFIG_ZRAM=m
+CONFIG_ZRAM_MEMORY_TRACKING=y
+CONFIG_ZRAM_WRITEBACK=y
diff --git a/chromeos/config/chromeos/base.config b/chromeos/config/chromeos/base.config
new file mode 100644
index 000000000000..c4dc66861561
--- /dev/null
+++ b/chromeos/config/chromeos/base.config
@@ -0,0 +1,477 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ANDROID_BINDER_IPC=y
+CONFIG_AUDIT=y
+CONFIG_BFQ_GROUP_IOSCHED=y
+CONFIG_BINFMT_MISC=y
+CONFIG_BLK_CGROUP=y
+# CONFIG_BLK_DEBUG_FS is not set
+CONFIG_BLK_DEV_DM=y
+CONFIG_BLK_DEV_INITRD=y
+CONFIG_BLK_DEV_IO_TRACE=y
+CONFIG_BLK_DEV_LOOP=y
+CONFIG_BLK_DEV_SD=y
+CONFIG_BLK_DEV_SR=m
+CONFIG_BOOTPARAM_HARDLOCKUP_PANIC=y
+CONFIG_BOOTPARAM_HUNG_TASK_PANIC=y
+CONFIG_BOOTPARAM_SOFTLOCKUP_PANIC=y
+CONFIG_BPF_JIT=y
+CONFIG_BPF_JIT_ALWAYS_ON=y
+CONFIG_BPF_SYSCALL=y
+CONFIG_BRIDGE=m
+CONFIG_BT=m
+CONFIG_BT_AOSPEXT=y
+CONFIG_BT_FEATURE_DEBUG=y
+CONFIG_BT_FEATURE_DEBUG_FUNC_NAMES=y
+CONFIG_BT_HCIBFUSB=m
+CONFIG_BT_HCIBTUSB=m
+CONFIG_BT_HCIBTUSB_AUTOSUSPEND=y
+CONFIG_BT_HCIVHCI=m
+CONFIG_BT_HIDP=m
+CONFIG_BT_MSFTEXT=y
+CONFIG_BT_RFCOMM=m
+CONFIG_CC_OPTIMIZE_FOR_SIZE=y
+CONFIG_CFG80211=m
+CONFIG_CFG80211_CERTIFICATION_ONUS=y
+CONFIG_CFG80211_DEBUGFS=y
+# CONFIG_CFG80211_REQUIRE_SIGNED_REGDB is not set
+CONFIG_CFS_BANDWIDTH=y
+CONFIG_CGROUPS=y
+CONFIG_CGROUP_BPF=y
+CONFIG_CGROUP_CPUACCT=y
+CONFIG_CGROUP_DEVICE=y
+CONFIG_CGROUP_FREEZER=y
+CONFIG_CGROUP_SCHED=y
+CONFIG_CHROME_PLATFORMS=y
+CONFIG_CLS_U32_MARK=y
+# CONFIG_COMPAT_BRK is not set
+CONFIG_CONNECTOR=y
+CONFIG_CPUSETS=y
+CONFIG_CPU_FREQ_GOV_USERSPACE=y
+CONFIG_CPU_FREQ_STAT=y
+CONFIG_CPU_IDLE_GOV_LADDER=y
+CONFIG_CPU_IDLE_GOV_MENU=y
+CONFIG_CPU_IDLE_GOV_TEO=y
+CONFIG_CRC7=m
+CONFIG_CROS_EC=y
+CONFIG_CROS_EC_PD_UPDATE=y
+CONFIG_CROS_EC_SENSORHUB=m
+CONFIG_CROS_EC_SPI=y
+CONFIG_CRYPTO_ARC4=y
+CONFIG_CRYPTO_DES=y
+CONFIG_CRYPTO_USER_API_HASH=m
+CONFIG_CRYPTO_USER_API_SKCIPHER=m
+CONFIG_DAX=y
+CONFIG_DEBUG_ATOMIC_SLEEP=y
+CONFIG_DEBUG_CREDENTIALS=y
+CONFIG_DEBUG_DEVRES=y
+CONFIG_DEBUG_INFO_DWARF5=y
+CONFIG_DEBUG_LIST=y
+# CONFIG_DEBUG_PREEMPT is not set
+CONFIG_DEBUG_SPINLOCK=y
+CONFIG_DEFAULT_HOSTNAME="localhost"
+CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
+CONFIG_DEVTMPFS=y
+CONFIG_DEVTMPFS_MOUNT=y
+CONFIG_DM_CRYPT=y
+CONFIG_DM_FLAKEY=m
+CONFIG_DM_INIT=y
+CONFIG_DM_INTEGRITY=m
+CONFIG_DM_THIN_PROVISIONING=y
+CONFIG_DM_VERITY=y
+CONFIG_DM_VERITY_CHROMEOS=y
+# CONFIG_DNOTIFY is not set
+CONFIG_DRM=y
+CONFIG_DRM_DP_AUX_CHARDEV=y
+# CONFIG_DRM_FBDEV_EMULATION is not set
+CONFIG_DRM_UDL=y
+CONFIG_ECRYPT_FS=y
+CONFIG_EMBEDDED=y
+CONFIG_ENCRYPTED_KEYS=y
+CONFIG_ESD_FS=y
+CONFIG_EXFAT_FS=m
+CONFIG_EXT4_FS=y
+CONFIG_EXT4_FS_POSIX_ACL=y
+CONFIG_EXT4_FS_SECURITY=y
+CONFIG_FANOTIFY=y
+CONFIG_FANOTIFY_ACCESS_PERMISSIONS=y
+CONFIG_FB=y
+CONFIG_FB_MODE_HELPERS=y
+CONFIG_FORTIFY_SOURCE=y
+CONFIG_FRAMEBUFFER_CONSOLE=y
+CONFIG_FRAMEBUFFER_CONSOLE_DETECT_PRIMARY=y
+CONFIG_FS_ENCRYPTION=y
+CONFIG_FS_VERITY=y
+CONFIG_FS_VERITY_BUILTIN_SIGNATURES=y
+CONFIG_FUNCTION_PROFILER=y
+CONFIG_FUNCTION_TRACER=y
+CONFIG_FUSE_FS=m
+CONFIG_GOOGLE_COREBOOT_TABLE=y
+CONFIG_GOOGLE_FIRMWARE=y
+CONFIG_GOOGLE_MEMCONSOLE_COREBOOT=y
+CONFIG_GOOGLE_VPD=y
+CONFIG_HARDENED_USERCOPY=y
+CONFIG_HFSPLUS_FS=m
+CONFIG_HIDRAW=y
+CONFIG_HID_APPLE=m
+CONFIG_HID_BATTERY_STRENGTH=y
+CONFIG_HID_CHERRY=m
+CONFIG_HID_CHICONY=m
+CONFIG_HID_HOLTEK=m
+CONFIG_HID_KENSINGTON=m
+CONFIG_HID_LOGITECH=m
+CONFIG_HID_LOGITECH_DJ=m
+CONFIG_HID_MAGICMOUSE=m
+CONFIG_HID_MICROSOFT=m
+CONFIG_HID_MULTITOUCH=y
+CONFIG_HID_NINTENDO=y
+CONFIG_HID_PLANTRONICS=m
+CONFIG_HID_PRIMAX=m
+CONFIG_HID_QUICKSTEP=m
+CONFIG_HID_SONY=m
+CONFIG_HID_THINGM=m
+CONFIG_HID_VIVALDI=m
+CONFIG_HID_WACOM=m
+CONFIG_HID_WIIMOTE=m
+CONFIG_HIGH_RES_TIMERS=y
+CONFIG_HIST_TRIGGERS=y
+CONFIG_HW_RANDOM=y
+CONFIG_I2C_STUB=m
+CONFIG_IIO_CROS_EC_SENSORS_CORE=m
+CONFIG_IIO_HRTIMER_TRIGGER=m
+CONFIG_IIO_SW_TRIGGER=m
+CONFIG_IIO_SYSFS_TRIGGER=m
+CONFIG_IKCONFIG=m
+CONFIG_IKCONFIG_PROC=y
+CONFIG_INET=y
+CONFIG_INET6_AH=m
+CONFIG_INET6_ESP=m
+CONFIG_INET_AH=y
+CONFIG_INET_DIAG_DESTROY=y
+CONFIG_INET_ESP=y
+CONFIG_INET_IPCOMP=y
+CONFIG_INET_UDP_DIAG=y
+CONFIG_INPUT_EVDEV=y
+CONFIG_INPUT_FF_MEMLESS=y
+CONFIG_INPUT_JOYDEV=m
+CONFIG_INPUT_JOYSTICK=y
+CONFIG_INPUT_MISC=y
+CONFIG_INPUT_TABLET=y
+CONFIG_INPUT_TOUCHSCREEN=y
+CONFIG_INPUT_UINPUT=m
+CONFIG_IOSCHED_BFQ=y
+CONFIG_IP6_NF_FILTER=y
+CONFIG_IP6_NF_IPTABLES=y
+CONFIG_IP6_NF_MANGLE=y
+CONFIG_IP6_NF_MATCH_IPV6HEADER=m
+CONFIG_IP6_NF_MATCH_RPFILTER=y
+CONFIG_IP6_NF_NAT=m
+CONFIG_IP6_NF_RAW=y
+CONFIG_IP6_NF_TARGET_MASQUERADE=m
+CONFIG_IP6_NF_TARGET_REJECT=m
+CONFIG_IPV6_MULTIPLE_TABLES=y
+CONFIG_IPV6_ROUTER_PREF=y
+CONFIG_IPV6_ROUTE_INFO=y
+CONFIG_IPV6_SIT=m
+CONFIG_IP_ADVANCED_ROUTER=y
+CONFIG_IP_MROUTE=y
+CONFIG_IP_MULTICAST=y
+CONFIG_IP_MULTIPLE_TABLES=y
+CONFIG_IP_NF_FILTER=y
+CONFIG_IP_NF_IPTABLES=y
+CONFIG_IP_NF_MANGLE=y
+CONFIG_IP_NF_NAT=y
+CONFIG_IP_NF_RAW=y
+CONFIG_IP_NF_SECURITY=y
+CONFIG_IP_NF_TARGET_MASQUERADE=m
+CONFIG_IP_NF_TARGET_REDIRECT=m
+CONFIG_IP_NF_TARGET_REJECT=y
+CONFIG_IP_PIMSM_V1=y
+CONFIG_IP_PIMSM_V2=y
+CONFIG_IP_ROUTE_MULTIPATH=y
+CONFIG_IP_ROUTE_VERBOSE=y
+CONFIG_ISO9660_FS=m
+CONFIG_JOLIET=y
+CONFIG_JOYSTICK_IFORCE=m
+CONFIG_JOYSTICK_IFORCE_USB=m
+CONFIG_JOYSTICK_XPAD=m
+CONFIG_JOYSTICK_XPAD_FF=y
+CONFIG_JOYSTICK_XPAD_LEDS=y
+CONFIG_KEYBOARD_CROS_EC=y
+CONFIG_KPROBES=y
+CONFIG_LEDS_CLASS=y
+# CONFIG_LEGACY_PTYS is not set
+CONFIG_LKDTM=y
+CONFIG_LOG_BUF_SHIFT=18
+CONFIG_LOW_MEM_NOTIFY=y
+CONFIG_LRU_GEN=y
+CONFIG_LRU_GEN_ENABLED=y
+CONFIG_MAC80211=m
+CONFIG_MAC80211_DEBUG_MENU=y
+CONFIG_MAC80211_HWSIM=m
+CONFIG_MAC80211_VERBOSE_DEBUG=y
+CONFIG_MAC_PARTITION=y
+CONFIG_MAGIC_SYSRQ=y
+CONFIG_MAGIC_SYSRQ_DEFAULT_ENABLE=0x1000
+CONFIG_MD=y
+CONFIG_MEDIA_CAMERA_SUPPORT=y
+CONFIG_MEDIA_PLATFORM_SUPPORT=y
+CONFIG_MEDIA_SUPPORT=y
+CONFIG_MEDIA_SUPPORT_FILTER=y
+CONFIG_MEDIA_USB_SUPPORT=y
+CONFIG_MMAP_NOEXEC_TAINT=0
+CONFIG_MMC=y
+CONFIG_MMC_BLOCK_MINORS=16
+CONFIG_MMC_SDHCI=y
+CONFIG_MMC_TEST=m
+CONFIG_MODULES=y
+CONFIG_MODULE_COMPRESS_GZIP=y
+CONFIG_MODULE_DECOMPRESS=y
+CONFIG_MODULE_FORCE_UNLOAD=y
+CONFIG_MODULE_UNLOAD=y
+CONFIG_MOUSE_CYAPA=y
+CONFIG_MOUSE_ELAN_I2C=y
+CONFIG_MWIFIEX=m
+CONFIG_NAMESPACES=y
+CONFIG_NET=y
+CONFIG_NETDEVICES=y
+CONFIG_NETFILTER=y
+CONFIG_NETFILTER_NETLINK_GLUE_CT=y
+CONFIG_NETFILTER_XT_MATCH_ADDRTYPE=y
+CONFIG_NETFILTER_XT_MATCH_BPF=y
+CONFIG_NETFILTER_XT_MATCH_CGROUP=m
+CONFIG_NETFILTER_XT_MATCH_COMMENT=y
+CONFIG_NETFILTER_XT_MATCH_CONNLIMIT=y
+CONFIG_NETFILTER_XT_MATCH_CONNMARK=y
+CONFIG_NETFILTER_XT_MATCH_CONNTRACK=y
+CONFIG_NETFILTER_XT_MATCH_DSCP=y
+CONFIG_NETFILTER_XT_MATCH_HASHLIMIT=y
+CONFIG_NETFILTER_XT_MATCH_HELPER=y
+CONFIG_NETFILTER_XT_MATCH_IPRANGE=y
+CONFIG_NETFILTER_XT_MATCH_LENGTH=y
+CONFIG_NETFILTER_XT_MATCH_LIMIT=y
+CONFIG_NETFILTER_XT_MATCH_MAC=y
+CONFIG_NETFILTER_XT_MATCH_MARK=y
+CONFIG_NETFILTER_XT_MATCH_OWNER=y
+CONFIG_NETFILTER_XT_MATCH_PKTTYPE=y
+CONFIG_NETFILTER_XT_MATCH_POLICY=y
+CONFIG_NETFILTER_XT_MATCH_QUOTA=y
+CONFIG_NETFILTER_XT_MATCH_SOCKET=y
+CONFIG_NETFILTER_XT_MATCH_STATE=y
+CONFIG_NETFILTER_XT_MATCH_STATISTIC=y
+CONFIG_NETFILTER_XT_MATCH_STRING=y
+CONFIG_NETFILTER_XT_MATCH_TIME=y
+CONFIG_NETFILTER_XT_MATCH_U32=y
+CONFIG_NETFILTER_XT_TARGET_CLASSIFY=y
+CONFIG_NETFILTER_XT_TARGET_CONNMARK=y
+CONFIG_NETFILTER_XT_TARGET_CONNSECMARK=y
+CONFIG_NETFILTER_XT_TARGET_CT=y
+CONFIG_NETFILTER_XT_TARGET_DSCP=y
+CONFIG_NETFILTER_XT_TARGET_IDLETIMER=y
+CONFIG_NETFILTER_XT_TARGET_MARK=y
+CONFIG_NETFILTER_XT_TARGET_NFLOG=y
+CONFIG_NETFILTER_XT_TARGET_NFQUEUE=y
+CONFIG_NETFILTER_XT_TARGET_REDIRECT=y
+CONFIG_NETFILTER_XT_TARGET_SECMARK=y
+CONFIG_NETFILTER_XT_TARGET_TCPMSS=y
+CONFIG_NETFILTER_XT_TARGET_TPROXY=y
+CONFIG_NETFILTER_XT_TARGET_TRACE=y
+CONFIG_NET_ACT_GACT=y
+CONFIG_NET_ACT_POLICE=y
+CONFIG_NET_CLS_ACT=y
+CONFIG_NET_CLS_U32=m
+CONFIG_NET_KEY=y
+CONFIG_NET_SCHED=y
+CONFIG_NET_SCH_CODEL=m
+CONFIG_NET_SCH_FQ_CODEL=m
+CONFIG_NET_SCH_HTB=m
+CONFIG_NET_SCH_INGRESS=m
+CONFIG_NET_SCH_NETEM=m
+CONFIG_NET_SCH_TBF=m
+CONFIG_NFS_FS=m
+CONFIG_NFS_V4=m
+CONFIG_NF_CONNTRACK=y
+CONFIG_NF_CONNTRACK_EVENTS=y
+CONFIG_NF_CONNTRACK_FTP=m
+CONFIG_NF_CONNTRACK_PPTP=y
+CONFIG_NF_CONNTRACK_SECMARK=y
+CONFIG_NF_CONNTRACK_TFTP=m
+CONFIG_NF_CONNTRACK_TIMEOUT=y
+CONFIG_NF_CT_NETLINK=y
+CONFIG_NF_CT_NETLINK_HELPER=y
+CONFIG_NF_CT_NETLINK_TIMEOUT=y
+# CONFIG_NF_CT_PROTO_DCCP is not set
+# CONFIG_NF_CT_PROTO_SCTP is not set
+# CONFIG_NF_CT_PROTO_UDPLITE is not set
+CONFIG_NF_REJECT_IPV6=y
+CONFIG_NL80211_TESTMODE=y
+CONFIG_NLS_ASCII=m
+CONFIG_NLS_CODEPAGE_437=m
+CONFIG_NLS_DEFAULT="utf8"
+CONFIG_NLS_ISO8859_1=m
+CONFIG_NO_HZ=y
+CONFIG_PACKET=y
+CONFIG_PANIC_ON_OOPS=y
+CONFIG_PANIC_TIMEOUT=-1
+CONFIG_PARTITION_ADVANCED=y
+CONFIG_PKGLIST=y
+CONFIG_PM_ADVANCED_DEBUG=y
+CONFIG_PM_DEBUG=y
+CONFIG_PM_WAKELOCKS=y
+CONFIG_PPP=y
+CONFIG_PPP_ASYNC=y
+CONFIG_PPP_MPPE=y
+CONFIG_PREEMPT=y
+CONFIG_PRINTK_TIME=y
+CONFIG_PROCESS_RECLAIM=y
+CONFIG_PROC_CHILDREN=y
+CONFIG_PROFILING=y
+CONFIG_PSI=y
+CONFIG_PSTORE_CONSOLE=y
+CONFIG_PSTORE_PMSG=y
+CONFIG_PSTORE_RAM=y
+CONFIG_QFMT_V2=y
+CONFIG_QUOTA=y
+CONFIG_RCU_EXPERT=y
+CONFIG_RCU_NOCB_CPU=y
+CONFIG_RCU_NOCB_CPU_DEFAULT_ALL=y
+# CONFIG_RCU_TRACE is not set
+# CONFIG_RD_BZIP2 is not set
+# CONFIG_RD_LZ4 is not set
+# CONFIG_RD_LZMA is not set
+# CONFIG_RD_LZO is not set
+# CONFIG_RD_ZSTD is not set
+CONFIG_RFKILL=y
+# CONFIG_RPCSEC_GSS_KRB5 is not set
+CONFIG_RT2800USB=m
+CONFIG_RT2800USB_RT3573=y
+CONFIG_RT2800USB_RT53XX=y
+CONFIG_RT2800USB_RT55XX=y
+CONFIG_RT2800USB_UNKNOWN=y
+CONFIG_RT2X00=m
+CONFIG_RTC_CLASS=y
+# CONFIG_RTC_SYSTOHC is not set
+CONFIG_SCHEDSTATS=y
+CONFIG_SCHED_TRACER=y
+CONFIG_SCSI_SCAN_ASYNC=y
+CONFIG_SCSI_SPI_ATTRS=y
+# CONFIG_SECTION_MISMATCH_WARN_ONLY is not set
+CONFIG_SECURITY=y
+CONFIG_SECURITY_DMESG_RESTRICT=y
+CONFIG_SECURITY_LANDLOCK=y
+CONFIG_SECURITY_LOADPIN=y
+CONFIG_SECURITY_LOADPIN_ENFORCE=y
+CONFIG_SECURITY_NETWORK=y
+CONFIG_SECURITY_SAFESETID=y
+CONFIG_SECURITY_SELINUX=y
+CONFIG_SECURITY_SELINUX_BOOTPARAM=y
+CONFIG_SECURITY_SELINUX_PERMISSIVE_DONTAUDIT=y
+CONFIG_SECURITY_YAMA=y
+CONFIG_SLAB_FREELIST_HARDENED=y
+CONFIG_SLAB_FREELIST_RANDOM=y
+CONFIG_SND=y
+CONFIG_SND_ALOOP=m
+CONFIG_SND_DUMMY=m
+CONFIG_SND_HRTIMER=m
+CONFIG_SND_SEQUENCER=m
+CONFIG_SND_SEQ_DUMMY=m
+CONFIG_SND_SOC=y
+# CONFIG_SND_SPI is not set
+CONFIG_SND_USB_AUDIO=m
+CONFIG_SOFT_WATCHDOG=m
+CONFIG_SOUND=y
+CONFIG_SPI=y
+CONFIG_SPI_SPIDEV=y
+CONFIG_SQUASHFS=y
+CONFIG_SQUASHFS_4K_DEVBLK_SIZE=y
+CONFIG_SQUASHFS_DECOMP_MULTI_PERCPU=y
+CONFIG_SQUASHFS_FILE_DIRECT=y
+CONFIG_SQUASHFS_LZ4=y
+CONFIG_SQUASHFS_LZO=y
+CONFIG_SQUASHFS_XATTR=y
+CONFIG_SQUASHFS_ZSTD=y
+CONFIG_STACKTRACE_BUILD_ID=y
+CONFIG_STAGING=y
+CONFIG_STATIC_USERMODEHELPER=y
+CONFIG_SW_SYNC=y
+CONFIG_SYN_COOKIES=y
+CONFIG_SYSVIPC=y
+CONFIG_TASKSTATS=y
+CONFIG_TASK_IO_ACCOUNTING=y
+CONFIG_TASK_XACCT=y
+CONFIG_TCG_TPM=y
+CONFIG_TCP_CONG_ADVANCED=y
+# CONFIG_TCP_CONG_BIC is not set
+# CONFIG_TCP_CONG_HTCP is not set
+CONFIG_TCP_CONG_LP=m
+# CONFIG_TCP_CONG_WESTWOOD is not set
+CONFIG_TCP_MD5SIG=y
+CONFIG_TEST_ASYNC_DRIVER_PROBE=m
+CONFIG_TEST_FIRMWARE=m
+CONFIG_TEST_LKM=m
+CONFIG_TEST_UDELAY=m
+CONFIG_TMPFS_POSIX_ACL=y
+CONFIG_TOUCHSCREEN_ATMEL_MXT=y
+CONFIG_TOUCHSCREEN_ELAN=y
+CONFIG_TOUCHSCREEN_USB_COMPOSITE=m
+CONFIG_TUN=m
+CONFIG_UDF_FS=m
+CONFIG_UDMABUF=y
+CONFIG_UHID=y
+CONFIG_UNICODE=y
+CONFIG_UNIX=y
+CONFIG_USB=y
+CONFIG_USBIP_CORE=m
+CONFIG_USBIP_VHCI_HCD=m
+CONFIG_USB_ACM=y
+CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
+CONFIG_USB_CONFIGFS=m
+CONFIG_USB_CONFIGFS_F_FS=y
+# CONFIG_USB_DEFAULT_PERSIST is not set
+CONFIG_USB_DUMMY_HCD=m
+CONFIG_USB_EHCI_HCD=y
+CONFIG_USB_HIDDEV=y
+CONFIG_USB_IPHETH=m
+CONFIG_USB_MASS_STORAGE=m
+CONFIG_USB_MON=y
+CONFIG_USB_NET_AQC111=m
+# CONFIG_USB_NET_CDC_SUBSET is not set
+CONFIG_USB_NET_DM9601=m
+CONFIG_USB_NET_MCS7830=m
+CONFIG_USB_NET_RNDIS_WLAN=m
+CONFIG_USB_NET_SMSC75XX=m
+CONFIG_USB_NET_SMSC95XX=m
+# CONFIG_USB_NET_ZAURUS is not set
+CONFIG_USB_PEGASUS=m
+CONFIG_USB_RTL8150=m
+CONFIG_USB_RTL8152=m
+CONFIG_USB_SERIAL=m
+CONFIG_USB_SERIAL_CH341=m
+CONFIG_USB_SERIAL_CP210X=m
+CONFIG_USB_SERIAL_FTDI_SIO=m
+CONFIG_USB_SERIAL_GENERIC=y
+CONFIG_USB_SERIAL_KEYSPAN=m
+CONFIG_USB_SERIAL_OPTION=m
+CONFIG_USB_SERIAL_OTI6858=m
+CONFIG_USB_SERIAL_PL2303=m
+CONFIG_USB_SERIAL_QUALCOMM=m
+CONFIG_USB_SERIAL_SIERRAWIRELESS=m
+CONFIG_USB_SERIAL_SIMPLE=m
+CONFIG_USB_STORAGE=y
+CONFIG_USB_UAS=y
+CONFIG_USB_VIDEO_CLASS=m
+# CONFIG_USB_VIDEO_CLASS_INPUT_EVDEV is not set
+CONFIG_USERFAULTFD=y
+CONFIG_USER_NS=y
+CONFIG_VETH=m
+CONFIG_VFAT_FS=m
+CONFIG_WATCHDOG=y
+CONFIG_WERROR=y
+CONFIG_WIREGUARD=m
+CONFIG_XFRM_INTERFACE=m
+CONFIG_XFRM_USER=y
+# CONFIG_XZ_DEC_IA64 is not set
+# CONFIG_XZ_DEC_POWERPC is not set
+# CONFIG_XZ_DEC_SPARC is not set
+CONFIG_ZISOFS=y
diff --git a/chromeos/config/chromeos/x86_64/chromeos-amd-stoneyridge.flavour.config b/chromeos/config/chromeos/x86_64/chromeos-amd-stoneyridge.flavour.config
new file mode 100644
index 000000000000..4dd3ba1dbf26
--- /dev/null
+++ b/chromeos/config/chromeos/x86_64/chromeos-amd-stoneyridge.flavour.config
@@ -0,0 +1,103 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ACERHDF=m
+CONFIG_AMDTEE=y
+CONFIG_AMD_IOMMU=y
+CONFIG_AMD_PMC=y
+CONFIG_ATH10K=m
+CONFIG_ATH10K_DEBUG=y
+CONFIG_ATH10K_DEBUGFS=y
+CONFIG_ATH10K_PCI=m
+CONFIG_ATH10K_TRACING=y
+CONFIG_ATH11K=m
+CONFIG_ATH11K_DEBUG=y
+CONFIG_ATH11K_DEBUGFS=y
+CONFIG_ATH11K_PCI=m
+CONFIG_ATH11K_TRACING=y
+CONFIG_BCMA=m
+CONFIG_BT_HCIBTUSB_MTK=y
+CONFIG_CFG80211_WEXT=y
+CONFIG_CPU_FREQ=y
+CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND=y
+CONFIG_CPU_FREQ_GOV_POWERSAVE=y
+# CONFIG_CPU_IBPB_ENTRY is not set
+# CONFIG_CPU_SUP_CENTAUR is not set
+# CONFIG_CPU_SUP_HYGON is not set
+# CONFIG_CPU_SUP_INTEL is not set
+# CONFIG_CPU_SUP_ZHAOXIN is not set
+CONFIG_CROS_EC_UART=y
+CONFIG_CRYPTO_DEV_CCP=y
+CONFIG_CRYPTO_DEV_CCP_DD=y
+CONFIG_CRYPTO_LZO=m
+CONFIG_CRYPTO_MICHAEL_MIC=m
+CONFIG_DEBUG_GPIO=y
+CONFIG_DEVTMPFS_SAFE=y
+CONFIG_DRM_AMDGPU=y
+CONFIG_DRM_AMDGPU_SI=y
+CONFIG_DRM_AMD_ACP=y
+CONFIG_DRM_AMD_DC_HDCP=y
+# CONFIG_EARLY_PRINTK is not set
+CONFIG_EDAC=y
+# CONFIG_EFIVAR_FS is not set
+CONFIG_GPIO_SYSFS=y
+CONFIG_HP_WMI=m
+CONFIG_HSA_AMD=y
+# CONFIG_HW_RANDOM_INTEL is not set
+CONFIG_I2C_DESIGNWARE_AMDPSP=y
+CONFIG_IRQ_REMAP=y
+CONFIG_KEYBOARD_GPIO=y
+CONFIG_MAILBOX=y
+CONFIG_MHI_BUS=y
+CONFIG_MHI_BUS_DEBUG=y
+CONFIG_MICROCODE_AMD=y
+CONFIG_MISC_RTSX_PCI=y
+CONFIG_MMC_REALTEK_PCI=y
+CONFIG_MT7921E=m
+CONFIG_MXM_WMI=y
+# CONFIG_NET_VENDOR_VERTEXCOM is not set
+CONFIG_PCC=y
+CONFIG_PHYLIB=y
+CONFIG_PINCTRL_AMD=y
+CONFIG_PROCESSOR_SELECT=y
+CONFIG_PTP_1588_CLOCK=m
+CONFIG_QRTR=y
+CONFIG_QRTR_MHI=y
+CONFIG_RTW88=m
+CONFIG_RTW88_8822CE=m
+CONFIG_RTW88_DEBUG=y
+CONFIG_RTW88_DEBUGFS=y
+CONFIG_RTW89_8852CE=m
+CONFIG_SCHED_CORE=y
+CONFIG_SCSI=y
+CONFIG_SERIAL_DEV_BUS=y
+CONFIG_SERIO_RAW=y
+CONFIG_SND_DESIGNWARE_I2S=m
+CONFIG_SND_HDA_GENERIC=m
+CONFIG_SND_SOC_AMD_ACP3x=m
+CONFIG_SND_SOC_AMD_ACP_COMMON=m
+CONFIG_SND_SOC_AMD_RENOIR=m
+CONFIG_SND_SOC_AMD_RENOIR_MACH=m
+CONFIG_SND_SOC_AMD_RV_RT5682_MACH=m
+CONFIG_SND_SOC_AMD_SOF_MACH=m
+# CONFIG_SND_SOC_INTEL_SST_TOPLEVEL is not set
+CONFIG_SND_SOC_SOF_ACPI=m
+CONFIG_SND_SOC_SOF_AMD_REMBRANDT=m
+CONFIG_SND_SOC_SOF_AMD_TOPLEVEL=m
+CONFIG_SND_SOC_SOF_PCI=m
+CONFIG_SND_SOC_SOF_TOPLEVEL=y
+CONFIG_SP5100_TCO=y
+CONFIG_SSB=m
+CONFIG_SSB_DRIVER_PCICORE=y
+CONFIG_SSB_SDIOHOST=y
+CONFIG_TCG_TIS_I2C_CR50=y
+CONFIG_TEE=y
+CONFIG_THERMAL_WRITABLE_TRIPS=y
+CONFIG_TMPFS=y
+CONFIG_TOUCHSCREEN_RM_TS=y
+CONFIG_WATCHDOG_SYSFS=y
+CONFIG_X86_AMD_FREQ_SENSITIVITY=y
+# CONFIG_X86_MCE_INTEL is not set
+CONFIG_ZRAM=m
+CONFIG_ZRAM_MEMORY_TRACKING=y
+CONFIG_ZRAM_WRITEBACK=y
diff --git a/chromeos/config/chromeos/x86_64/chromeos-intel-denverton.flavour.config b/chromeos/config/chromeos/x86_64/chromeos-intel-denverton.flavour.config
new file mode 100644
index 000000000000..19db8888a0be
--- /dev/null
+++ b/chromeos/config/chromeos/x86_64/chromeos-intel-denverton.flavour.config
@@ -0,0 +1,62 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ATA=y
+CONFIG_ATA_GENERIC=y
+CONFIG_ATA_PIIX=y
+CONFIG_B43=m
+CONFIG_B43_SDIO=y
+CONFIG_CFG80211_WEXT=y
+CONFIG_CPU_FREQ_GOV_POWERSAVE=y
+# CONFIG_CPU_SUP_CENTAUR is not set
+# CONFIG_CPU_SUP_HYGON is not set
+# CONFIG_CPU_SUP_ZHAOXIN is not set
+CONFIG_CRYPTO_LZO=m
+CONFIG_DEVTMPFS_SAFE=y
+CONFIG_DMADEVICES=y
+CONFIG_DRM_I915=y
+CONFIG_DW_DMAC=m
+CONFIG_EARLY_PRINTK_DBGP=y
+# CONFIG_EFIVAR_FS is not set
+CONFIG_IGB=y
+CONFIG_INT340X_THERMAL=y
+CONFIG_INTEL_IDLE=y
+CONFIG_INTEL_IDMA64=y
+CONFIG_INTEL_MENLOW=y
+CONFIG_INTEL_PMC_CORE=y
+CONFIG_INTEL_SOC_DTS_THERMAL=y
+CONFIG_IRQ_REMAP=y
+CONFIG_ITCO_VENDOR_SUPPORT=y
+CONFIG_ITCO_WDT=y
+CONFIG_IWLDVM=m
+CONFIG_IWLMVM=m
+CONFIG_IWLWIFI=m
+CONFIG_IWLWIFI_DEBUGFS=y
+CONFIG_IXGBE=y
+CONFIG_MFD_INTEL_LPSS_ACPI=y
+CONFIG_MFD_INTEL_LPSS_PCI=y
+# CONFIG_MOUSE_PS2_TRACKPOINT is not set
+CONFIG_MTD=m
+CONFIG_MTD_SPI_NOR=m
+CONFIG_NETWORK_PHY_TIMESTAMPING=y
+# CONFIG_NET_VENDOR_VERTEXCOM is not set
+CONFIG_PCI_MSI=y
+CONFIG_PROCESSOR_SELECT=y
+CONFIG_RTL8187=m
+CONFIG_RTL8192CU=m
+# CONFIG_RTLWIFI_DEBUG is not set
+CONFIG_SATA_AHCI=y
+CONFIG_SATA_MOBILE_LPM_POLICY=3
+# CONFIG_SATA_PMP is not set
+CONFIG_SCHED_CORE=y
+CONFIG_SKGE=m
+CONFIG_SKY2=m
+CONFIG_SPI_INTEL_PCI=m
+CONFIG_TIGON3=m
+CONFIG_X86_INTEL_LPSS=y
+# CONFIG_X86_MCE_AMD is not set
+CONFIG_X86_PKG_TEMP_THERMAL=y
+CONFIG_X86_X2APIC=y
+CONFIG_ZRAM=m
+CONFIG_ZRAM_MEMORY_TRACKING=y
+CONFIG_ZRAM_WRITEBACK=y
diff --git a/chromeos/config/chromeos/x86_64/chromeos-intel-pineview.flavour.config b/chromeos/config/chromeos/x86_64/chromeos-intel-pineview.flavour.config
new file mode 100644
index 000000000000..ffd9940fcca8
--- /dev/null
+++ b/chromeos/config/chromeos/x86_64/chromeos-intel-pineview.flavour.config
@@ -0,0 +1,192 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ACPI_DPTF=y
+CONFIG_ATA=y
+CONFIG_ATA_GENERIC=y
+CONFIG_ATA_PIIX=y
+CONFIG_BATTERY_SBS=m
+CONFIG_BT_HCIBTUSB_MTK=y
+CONFIG_CEC_CROS_EC=m
+CONFIG_CFG80211_WEXT=y
+CONFIG_CHARGER_WILCO=m
+CONFIG_CHROMEOS_PRIVACY_SCREEN=y
+CONFIG_CPU_FREQ_GOV_ONDEMAND=y
+CONFIG_CROS_EC_ISHTP=m
+CONFIG_CRYPTO_LZO=m
+CONFIG_DEBUG_GPIO=y
+CONFIG_DEVTMPFS_SAFE=y
+CONFIG_DMADEVICES=y
+CONFIG_DPTF_PCH_FIVR=y
+CONFIG_DRM_I915=y
+CONFIG_DRM_I915_FORCE_PROBE="*"
+CONFIG_DRM_I915_PXP=y
+CONFIG_DW_DMAC=y
+CONFIG_EARLY_PRINTK_DBGP=y
+CONFIG_EEPROM_AT24=m
+# CONFIG_EFIVAR_FS is not set
+CONFIG_GPIO_SYSFS=y
+CONFIG_GPIO_TPS68470=y
+CONFIG_HOTPLUG_PCI_ACPI=y
+CONFIG_HP_WMI=m
+# CONFIG_HW_RANDOM_AMD is not set
+CONFIG_I2C_CROS_EC_TUNNEL=m
+CONFIG_IGB=m
+CONFIG_IGC=m
+CONFIG_INT340X_THERMAL=y
+CONFIG_INTEL_HFI_THERMAL=y
+CONFIG_INTEL_IDLE=y
+CONFIG_INTEL_IDMA64=y
+CONFIG_INTEL_IOMMU=y
+# CONFIG_INTEL_IOMMU_DEFAULT_ON is not set
+CONFIG_INTEL_IOMMU_SVM=y
+CONFIG_INTEL_ISH_FIRMWARE_DOWNLOADER=m
+CONFIG_INTEL_ISH_HID=m
+CONFIG_INTEL_MEI_HDCP=m
+CONFIG_INTEL_MEI_PXP=y
+CONFIG_INTEL_MENLOW=m
+CONFIG_INTEL_PMC_CORE=y
+CONFIG_INTEL_PUNIT_IPC=y
+CONFIG_INTEL_SCU_PLATFORM=y
+CONFIG_INTEL_SKL_INT3472=y
+CONFIG_INTEL_SOC_DTS_THERMAL=y
+CONFIG_INTEL_TELEMETRY=y
+CONFIG_INTEL_TURBO_MAX_3=y
+CONFIG_IRQ_REMAP=y
+CONFIG_ITCO_VENDOR_SUPPORT=y
+CONFIG_ITCO_WDT=y
+CONFIG_IWLDVM=m
+CONFIG_IWLMVM=m
+CONFIG_IWLWIFI=m
+CONFIG_IWLWIFI_DEBUGFS=y
+CONFIG_KEYBOARD_GPIO=y
+CONFIG_MFD_INTEL_LPSS_ACPI=y
+CONFIG_MFD_INTEL_LPSS_PCI=y
+CONFIG_MFD_INTEL_PMC_BXT=y
+CONFIG_MISC_RTSX_PCI=m
+CONFIG_MMC_REALTEK_PCI=m
+# CONFIG_MOUSE_PS2_TRACKPOINT is not set
+CONFIG_MT7921E=m
+CONFIG_MTK_T7XX=m
+# CONFIG_NET_VENDOR_VERTEXCOM is not set
+CONFIG_NR_CPUS=24
+CONFIG_PCI_IOV=y
+CONFIG_PCI_MSI=y
+CONFIG_PCI_PF_STUB=m
+CONFIG_PCI_STUB=m
+CONFIG_PHYLIB=y
+CONFIG_PINCTRL_ALDERLAKE=y
+CONFIG_PINCTRL_BAYTRAIL=y
+CONFIG_PINCTRL_BROXTON=y
+CONFIG_PINCTRL_CANNONLAKE=y
+CONFIG_PINCTRL_CHERRYVIEW=y
+CONFIG_PINCTRL_GEMINILAKE=y
+CONFIG_PINCTRL_ICELAKE=y
+CONFIG_PINCTRL_JASPERLAKE=y
+CONFIG_PINCTRL_METEORLAKE=y
+CONFIG_PINCTRL_SUNRISEPOINT=y
+CONFIG_PINCTRL_TIGERLAKE=y
+CONFIG_PTP_1588_CLOCK=m
+CONFIG_PUNIT_ATOM_DEBUG=y
+CONFIG_REGULATOR=y
+# CONFIG_RT2800PCI_RT33XX is not set
+# CONFIG_RT2800PCI_RT35XX is not set
+# CONFIG_RT2800PCI_RT53XX is not set
+CONFIG_RTW88=m
+CONFIG_RTW88_8822BE=m
+CONFIG_RTW88_8822CE=m
+CONFIG_RTW88_DEBUG=y
+CONFIG_RTW88_DEBUGFS=y
+CONFIG_SATA_AHCI=y
+CONFIG_SATA_MOBILE_LPM_POLICY=3
+# CONFIG_SATA_PMP is not set
+CONFIG_SCHED_CORE=y
+CONFIG_SCSI_UFSHCD=y
+CONFIG_SCSI_UFSHCD_PCI=y
+CONFIG_SCSI_UFS_BSG=y
+CONFIG_SENSORS_ISL29018=m
+CONFIG_SND_HDA_CODEC_ANALOG=m
+CONFIG_SND_HDA_CODEC_CA0110=m
+CONFIG_SND_HDA_CODEC_CA0132=m
+CONFIG_SND_HDA_CODEC_CIRRUS=m
+CONFIG_SND_HDA_CODEC_CMEDIA=m
+CONFIG_SND_HDA_CODEC_CONEXANT=m
+CONFIG_SND_HDA_CODEC_REALTEK=m
+CONFIG_SND_HDA_CODEC_SI3054=m
+CONFIG_SND_HDA_CODEC_SIGMATEL=m
+CONFIG_SND_HDA_CODEC_VIA=m
+CONFIG_SND_HDA_POWER_SAVE_DEFAULT=15
+CONFIG_SND_SOC_INTEL_BYTCR_RT5640_MACH=m
+CONFIG_SND_SOC_INTEL_CHT_BSW_MAX98090_TI_MACH=m
+CONFIG_SND_SOC_INTEL_CHT_BSW_RT5645_MACH=m
+CONFIG_SND_SOC_INTEL_CHT_BSW_RT5672_MACH=m
+CONFIG_SND_SOC_INTEL_CML_LP_DA7219_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_GLK_DA7219_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_GLK_RT5682_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_KBL_DA7219_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_KBL_DA7219_MAX98927_MACH=m
+CONFIG_SND_SOC_INTEL_KBL_RT5660_MACH=m
+CONFIG_SND_SOC_INTEL_KBL_RT5663_MAX98927_MACH=m
+CONFIG_SND_SOC_INTEL_KBL_RT5663_RT5514_MAX98927_MACH=m
+CONFIG_SND_SOC_INTEL_SKL_NAU88L25_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_SKL_NAU88L25_SSM4567_MACH=m
+CONFIG_SND_SOC_INTEL_SKL_RT286_MACH=m
+CONFIG_SND_SOC_INTEL_SKYLAKE=m
+CONFIG_SND_SOC_INTEL_SKYLAKE_HDAUDIO_CODEC=y
+CONFIG_SND_SOC_INTEL_SOF_CML_RT1011_RT5682_MACH=m
+CONFIG_SND_SOC_INTEL_SOF_CS42L42_MACH=m
+CONFIG_SND_SOC_INTEL_SOF_DA7219_MAX98373_MACH=m
+CONFIG_SND_SOC_INTEL_SOF_NAU8825_MACH=m
+CONFIG_SND_SOC_INTEL_SOF_RT5682_MACH=m
+CONFIG_SND_SOC_INTEL_SOUNDWIRE_SOF_MACH=m
+CONFIG_SND_SOC_INTEL_USER_FRIENDLY_LONG_NAMES=y
+CONFIG_SND_SOC_SOF_ACPI=m
+CONFIG_SND_SOC_SOF_HDA_AUDIO_CODEC=y
+CONFIG_SND_SOC_SOF_HDA_LINK=y
+CONFIG_SND_SOC_SOF_INTEL_TOPLEVEL=y
+CONFIG_SND_SOC_SOF_PCI=m
+CONFIG_SND_SOC_SOF_TOPLEVEL=y
+CONFIG_SND_SOC_TS3A227E=y
+CONFIG_SND_SST_ATOM_HIFI2_PLATFORM_ACPI=m
+CONFIG_SOUNDWIRE=m
+CONFIG_STAGING_MEDIA=y
+CONFIG_SX9310=m
+CONFIG_SX9324=m
+CONFIG_TCG_TIS_I2C_CR50=y
+CONFIG_TOUCHSCREEN_RM_TS=y
+CONFIG_TPS68470_PMIC_OPREGION=y
+CONFIG_TYPEC_MUX_INTEL_PMC=m
+CONFIG_USB4=y
+CONFIG_USB4_DEBUGFS_WRITE=y
+CONFIG_USB4_DMA_TEST=m
+CONFIG_USB4_NET=m
+CONFIG_VFIO=m
+CONFIG_VFIO_PCI=m
+CONFIG_VIDEO_DW9714=m
+CONFIG_VIDEO_DW9768=m
+CONFIG_VIDEO_HI556=m
+CONFIG_VIDEO_IMX208=m
+CONFIG_VIDEO_INTEL_IPU6=m
+CONFIG_VIDEO_IPU3_CIO2=m
+CONFIG_VIDEO_IPU3_IMGU=m
+CONFIG_VIDEO_OV13858=m
+CONFIG_VIDEO_OV13B10=m
+CONFIG_VIDEO_OV2740=m
+CONFIG_VIDEO_OV5670=m
+CONFIG_VIDEO_OV5675=m
+CONFIG_VIDEO_OV8856=m
+CONFIG_VIDEO_OV9734=m
+CONFIG_WILCO_EC=m
+CONFIG_WILCO_EC_CHARGE_SCHEDULE=m
+CONFIG_WILCO_EC_DEBUGFS=m
+CONFIG_WILCO_EC_EVENTS=m
+CONFIG_WILCO_EC_TELEMETRY=m
+CONFIG_WMI_BMOF=m
+CONFIG_WWAN=y
+CONFIG_X86_INTEL_LPSS=y
+# CONFIG_X86_MCE_AMD is not set
+CONFIG_X86_PKG_TEMP_THERMAL=y
+CONFIG_X86_X2APIC=y
+CONFIG_ZRAM=m
+CONFIG_ZRAM_MEMORY_TRACKING=y
+CONFIG_ZRAM_WRITEBACK=y
diff --git a/chromeos/config/chromeos/x86_64/chromeos-x86_64-reven.flavour.config b/chromeos/config/chromeos/x86_64/chromeos-x86_64-reven.flavour.config
new file mode 100644
index 000000000000..a75824415964
--- /dev/null
+++ b/chromeos/config/chromeos/x86_64/chromeos-x86_64-reven.flavour.config
@@ -0,0 +1,274 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ACERHDF=m
+CONFIG_ACER_WMI=m
+CONFIG_ACPI_CMPC=m
+CONFIG_ACPI_SBS=m
+CONFIG_ACPI_TOSHIBA=m
+CONFIG_APPLE_GMUX=m
+CONFIG_AR5523=m
+CONFIG_ASUS_LAPTOP=m
+CONFIG_ASUS_NB_WMI=m
+CONFIG_ASUS_WIRELESS=m
+CONFIG_ASUS_WMI=m
+CONFIG_ATA=y
+CONFIG_ATA_GENERIC=y
+CONFIG_ATA_PIIX=y
+CONFIG_ATH10K=m
+CONFIG_ATH10K_PCI=m
+CONFIG_ATH10K_SDIO=m
+CONFIG_ATH10K_USB=m
+CONFIG_ATH5K=m
+CONFIG_ATH6KL=m
+CONFIG_ATH6KL_SDIO=m
+CONFIG_ATH6KL_USB=m
+CONFIG_ATH9K_AHB=y
+CONFIG_ATH9K_HTC=m
+CONFIG_ATL1=m
+CONFIG_ATL1C=m
+CONFIG_ATL1E=m
+CONFIG_ATL2=m
+CONFIG_B43=m
+CONFIG_B43LEGACY=m
+CONFIG_B43_SDIO=y
+CONFIG_BATTERY_SBS=m
+CONFIG_BRCMSMAC=m
+CONFIG_BROADCOM_PHY=m
+CONFIG_COMPAL_LAPTOP=m
+CONFIG_CPU_FREQ_GOV_POWERSAVE=y
+# CONFIG_CPU_SUP_CENTAUR is not set
+# CONFIG_CPU_SUP_HYGON is not set
+# CONFIG_CPU_SUP_ZHAOXIN is not set
+CONFIG_CRYPTO_DEV_VIRTIO=m
+CONFIG_DEBUG_GPIO=y
+CONFIG_DMADEVICES=y
+CONFIG_DRM_AMDGPU=m
+CONFIG_DRM_GMA500=m
+# CONFIG_DRM_I2C_CH7006 is not set
+# CONFIG_DRM_I2C_SIL164 is not set
+CONFIG_DRM_I915=m
+CONFIG_DRM_I915_FORCE_PROBE="*"
+CONFIG_DRM_NOUVEAU=m
+CONFIG_DRM_RADEON=m
+CONFIG_DRM_VIRTIO_GPU=y
+CONFIG_DW_DMAC=m
+CONFIG_EARLY_PRINTK_DBGP=y
+CONFIG_EEEPC_LAPTOP=m
+CONFIG_EEEPC_WMI=m
+CONFIG_EFIVAR_FS=y
+CONFIG_EFI_MIXED=y
+CONFIG_EFI_STUB=y
+CONFIG_FORCEDETH=m
+CONFIG_FUJITSU_LAPTOP=m
+CONFIG_FUJITSU_TABLET=m
+CONFIG_GPIO_SYSFS=y
+CONFIG_HDMI_LPE_AUDIO=m
+CONFIG_HID_ACCUTOUCH=m
+CONFIG_HID_ALPS=m
+CONFIG_HID_ASUS=m
+CONFIG_HID_CMEDIA=m
+CONFIG_HID_CORSAIR=m
+CONFIG_HID_ELO=m
+CONFIG_HID_ITE=m
+CONFIG_HID_KYE=m
+CONFIG_HID_LCPOWER=m
+CONFIG_HID_LENOVO=m
+CONFIG_HID_NTI=m
+CONFIG_HID_SENSOR_ACCEL_3D=m
+CONFIG_HID_SENSOR_ALS=m
+CONFIG_HID_SENSOR_DEVICE_ROTATION=m
+CONFIG_HID_SENSOR_GYRO_3D=m
+CONFIG_HID_SENSOR_HUB=m
+CONFIG_HID_SENSOR_HUMIDITY=m
+CONFIG_HID_SENSOR_INCLINOMETER_3D=m
+CONFIG_HID_SENSOR_MAGNETOMETER_3D=m
+CONFIG_HID_SENSOR_PRESS=m
+CONFIG_HID_SENSOR_PROX=m
+CONFIG_HID_SENSOR_TEMP=m
+CONFIG_HOTPLUG_PCI_ACPI=y
+CONFIG_HP_ACCEL=m
+CONFIG_HP_WMI=m
+CONFIG_I2C_CROS_EC_TUNNEL=m
+CONFIG_IBM_RTL=m
+CONFIG_IDEAPAD_LAPTOP=m
+CONFIG_IGB=m
+CONFIG_IGC=m
+CONFIG_INTEL_HID_EVENT=m
+CONFIG_INTEL_IDLE=y
+CONFIG_INTEL_IDMA64=y
+CONFIG_INTEL_INT0002_VGPIO=m
+CONFIG_INTEL_IPS=m
+CONFIG_INTEL_ISH_HID=m
+CONFIG_INTEL_MEI_ME=m
+CONFIG_INTEL_MEI_TXE=m
+CONFIG_INTEL_MENLOW=m
+CONFIG_INTEL_OAKTRAIL=m
+CONFIG_INTEL_PMC_CORE=y
+CONFIG_INTEL_PUNIT_IPC=m
+CONFIG_INTEL_RST=m
+CONFIG_INTEL_SCU_PLATFORM=y
+CONFIG_INTEL_SMARTCONNECT=m
+CONFIG_INTEL_SOC_DTS_THERMAL=y
+CONFIG_INTEL_TURBO_MAX_3=y
+CONFIG_IWL3945=m
+CONFIG_IWL4965=m
+CONFIG_KEYBOARD_GPIO=y
+CONFIG_LOCK_DOWN_KERNEL_FORCE_INTEGRITY=y
+CONFIG_LSM="landlock,lockdown,yama,loadpin,safesetid,integrity,chromiumos,selinux"
+CONFIG_MARVELL_PHY=m
+CONFIG_MAXLINEAR_GPHY=m
+CONFIG_MFD_INTEL_LPSS_ACPI=y
+CONFIG_MFD_INTEL_LPSS_PCI=y
+CONFIG_MICROCODE_AMD=y
+CONFIG_MISC_RTSX_PCI=m
+CONFIG_MLX_PLATFORM=m
+CONFIG_MMC_CB710=m
+CONFIG_MMC_REALTEK_PCI=m
+CONFIG_MMC_USDHI6ROL0=m
+CONFIG_MMC_USHC=m
+CONFIG_MMC_VIA_SDMMC=m
+CONFIG_MMC_VUB300=m
+CONFIG_MODULE_SIG_FORCE=y
+CONFIG_MODULE_SIG_SHA256=y
+CONFIG_MOUSE_BCM5974=m
+CONFIG_MOUSE_ELAN_I2C_SMBUS=y
+CONFIG_MOUSE_GPIO=m
+CONFIG_MOUSE_PS2_ELANTECH=y
+CONFIG_MOUSE_PS2_SENTELIC=y
+CONFIG_MOUSE_PS2_TOUCHKIT=y
+CONFIG_MOUSE_SERIAL=m
+CONFIG_MOUSE_SYNAPTICS_I2C=m
+CONFIG_MOUSE_SYNAPTICS_USB=m
+CONFIG_MSI_LAPTOP=m
+CONFIG_MSI_WMI=m
+CONFIG_MT7601U=m
+CONFIG_MT76x0U=m
+CONFIG_MT76x2E=m
+CONFIG_MT76x2U=m
+CONFIG_MWIFIEX_SDIO=m
+CONFIG_MWIFIEX_USB=m
+CONFIG_MWL8K=m
+CONFIG_PANASONIC_LAPTOP=m
+CONFIG_PCI_MSI=y
+CONFIG_PHYLIB=y
+CONFIG_PINCTRL_AMD=y
+CONFIG_PINCTRL_BAYTRAIL=y
+CONFIG_PINCTRL_BROXTON=y
+CONFIG_PINCTRL_CANNONLAKE=y
+CONFIG_PINCTRL_CHERRYVIEW=y
+CONFIG_PINCTRL_GEMINILAKE=y
+CONFIG_PINCTRL_ICELAKE=y
+CONFIG_PINCTRL_JASPERLAKE=y
+CONFIG_PINCTRL_SUNRISEPOINT=y
+CONFIG_PINCTRL_TIGERLAKE=y
+CONFIG_PROCESSOR_SELECT=y
+CONFIG_R8188EU=m
+CONFIG_R8712U=m
+CONFIG_RESET_ATTACK_MITIGATION=y
+CONFIG_RMI4_I2C=m
+CONFIG_RMI4_SMB=m
+CONFIG_RMI4_SPI=m
+CONFIG_RT2400PCI=m
+CONFIG_RT2500PCI=m
+CONFIG_RT2500USB=m
+CONFIG_RT61PCI=m
+CONFIG_RT73USB=m
+CONFIG_RTL8180=m
+CONFIG_RTL8187=m
+CONFIG_RTL8188EE=m
+CONFIG_RTL8192CE=m
+CONFIG_RTL8192CU=m
+CONFIG_RTL8192DE=m
+CONFIG_RTL8192EE=m
+CONFIG_RTL8192SE=m
+CONFIG_RTL8192U=m
+CONFIG_RTL8723AE=m
+CONFIG_RTL8723BE=m
+CONFIG_RTL8723BS=m
+CONFIG_RTL8821AE=m
+# CONFIG_RTLWIFI_DEBUG is not set
+CONFIG_RTW88=m
+CONFIG_RTW88_8821CE=m
+CONFIG_RTW88_8822BE=m
+CONFIG_RTW88_8822CE=m
+CONFIG_RTW88_DEBUG=y
+CONFIG_RTW88_DEBUGFS=y
+CONFIG_SAMSUNG_LAPTOP=m
+CONFIG_SATA_AHCI=y
+CONFIG_SATA_MOBILE_LPM_POLICY=3
+# CONFIG_SATA_PMP is not set
+CONFIG_SCSI_VIRTIO=y
+CONFIG_SECURITY_LOCKDOWN_LSM=y
+CONFIG_SECURITY_LOCKDOWN_LSM_EARLY=y
+CONFIG_SENSORS_ACPI_POWER=m
+CONFIG_SENSORS_FAM15H_POWER=m
+CONFIG_SENSORS_HDAPS=m
+CONFIG_SENSORS_K10TEMP=m
+CONFIG_SKGE=m
+CONFIG_SKY2=m
+CONFIG_SND_HDA_CODEC_ANALOG=m
+CONFIG_SND_HDA_CODEC_CA0110=m
+CONFIG_SND_HDA_CODEC_CA0132=m
+CONFIG_SND_HDA_CODEC_CIRRUS=m
+CONFIG_SND_HDA_CODEC_CMEDIA=m
+CONFIG_SND_HDA_CODEC_CONEXANT=m
+CONFIG_SND_HDA_CODEC_REALTEK=m
+CONFIG_SND_HDA_CODEC_SI3054=m
+CONFIG_SND_HDA_CODEC_SIGMATEL=m
+CONFIG_SND_HDA_CODEC_VIA=m
+CONFIG_SND_HDA_POWER_SAVE_DEFAULT=15
+CONFIG_SND_SOC_ADAU7002=m
+CONFIG_SND_SOC_AMD_RENOIR=m
+CONFIG_SND_SOC_AMD_RENOIR_MACH=m
+CONFIG_SND_SOC_INTEL_BXT_DA7219_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_BXT_RT298_MACH=m
+CONFIG_SND_SOC_INTEL_BYTCR_RT5640_MACH=m
+CONFIG_SND_SOC_INTEL_BYTCR_RT5651_MACH=m
+CONFIG_SND_SOC_INTEL_BYT_CHT_DA7213_MACH=m
+CONFIG_SND_SOC_INTEL_BYT_CHT_ES8316_MACH=m
+CONFIG_SND_SOC_INTEL_BYT_CHT_NOCODEC_MACH=m
+CONFIG_SND_SOC_INTEL_CHT_BSW_RT5645_MACH=m
+CONFIG_SND_SOC_INTEL_CML_H=m
+CONFIG_SND_SOC_INTEL_CML_LP=m
+CONFIG_SND_SOC_INTEL_CML_LP_DA7219_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_KBL_DA7219_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_KBL_DA7219_MAX98927_MACH=m
+CONFIG_SND_SOC_INTEL_KBL_RT5660_MACH=m
+CONFIG_SND_SOC_INTEL_KBL_RT5663_MAX98927_MACH=m
+CONFIG_SND_SOC_INTEL_SKL_HDA_DSP_GENERIC_MACH=m
+CONFIG_SND_SOC_INTEL_SKYLAKE=m
+CONFIG_SND_SOC_SOF_ACPI=m
+CONFIG_SND_SOC_SOF_HDA_AUDIO_CODEC=y
+CONFIG_SND_SOC_SOF_HDA_LINK=y
+CONFIG_SND_SOC_SOF_INTEL_TOPLEVEL=y
+CONFIG_SND_SOC_SOF_PCI=m
+CONFIG_SND_SOC_SOF_TOPLEVEL=y
+CONFIG_SND_SOC_SSM4567=m
+CONFIG_SND_SOC_TS3A227E=y
+CONFIG_SONYPI_COMPAT=y
+CONFIG_SONY_LAPTOP=m
+CONFIG_SOUNDWIRE=m
+CONFIG_STMMAC_ETH=m
+CONFIG_SURFACE3_WMI=m
+CONFIG_SURFACE_PRO3_BUTTON=m
+CONFIG_SX9310=m
+CONFIG_THINKPAD_ACPI=m
+CONFIG_TIGON3=m
+CONFIG_TOPSTAR_LAPTOP=m
+CONFIG_TOSHIBA_BT_RFKILL=m
+CONFIG_TOUCHSCREEN_RM_TS=y
+CONFIG_USB_HCD_BCMA=m
+CONFIG_USB_HCD_SSB=m
+CONFIG_USB_LEDS_TRIGGER_USBPORT=m
+CONFIG_USB_OHCI_HCD=y
+CONFIG_USB_R8A66597_HCD=m
+CONFIG_VIRTIO_BLK=m
+CONFIG_VIRTIO_NET=m
+CONFIG_VIRTIO_PCI=y
+CONFIG_VMD=y
+CONFIG_WIL6210=m
+CONFIG_X86_INTEL_LPSS=y
+# CONFIG_X86_MCE_AMD is not set
+CONFIG_X86_PKG_TEMP_THERMAL=y
+# CONFIG_X86_VERBOSE_BOOTUP is not set
diff --git a/chromeos/config/chromeos/x86_64/chromiumos-x86_64.flavour.config b/chromeos/config/chromeos/x86_64/chromiumos-x86_64.flavour.config
new file mode 100644
index 000000000000..e94976ed199a
--- /dev/null
+++ b/chromeos/config/chromeos/x86_64/chromiumos-x86_64.flavour.config
@@ -0,0 +1,113 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ACERHDF=m
+CONFIG_ATA=y
+CONFIG_ATA_GENERIC=y
+CONFIG_ATA_PIIX=y
+CONFIG_B43=m
+CONFIG_B43_SDIO=y
+CONFIG_BATTERY_SBS=m
+CONFIG_CFG80211_WEXT=y
+CONFIG_CPU_FREQ_GOV_POWERSAVE=y
+# CONFIG_CPU_SUP_CENTAUR is not set
+# CONFIG_CPU_SUP_HYGON is not set
+# CONFIG_CPU_SUP_ZHAOXIN is not set
+CONFIG_CRYPTO_DEV_VIRTIO=m
+CONFIG_CRYPTO_LZO=m
+CONFIG_DEBUG_GPIO=y
+CONFIG_DEVTMPFS_SAFE=y
+CONFIG_DMADEVICES=y
+CONFIG_DRM_GMA500=y
+# CONFIG_DRM_I2C_CH7006 is not set
+# CONFIG_DRM_I2C_SIL164 is not set
+CONFIG_DRM_I915=y
+CONFIG_DRM_I915_FORCE_PROBE="*"
+CONFIG_DRM_NOUVEAU=y
+CONFIG_DRM_RADEON=m
+CONFIG_DRM_VIRTIO_GPU=y
+CONFIG_DW_DMAC=m
+CONFIG_EARLY_PRINTK_DBGP=y
+# CONFIG_EFIVAR_FS is not set
+CONFIG_GPIO_SYSFS=y
+CONFIG_HOTPLUG_PCI_ACPI=y
+CONFIG_HP_WMI=m
+CONFIG_I2C_CROS_EC_TUNNEL=m
+CONFIG_INTEL_IDLE=y
+CONFIG_INTEL_IDMA64=y
+CONFIG_INTEL_MENLOW=y
+CONFIG_INTEL_PMC_CORE=y
+CONFIG_INTEL_SCU_PLATFORM=y
+CONFIG_INTEL_SOC_DTS_THERMAL=y
+CONFIG_KEYBOARD_GPIO=y
+CONFIG_MFD_INTEL_LPSS_ACPI=y
+CONFIG_MFD_INTEL_LPSS_PCI=y
+CONFIG_MICROCODE_AMD=y
+CONFIG_MOUSE_BCM5974=m
+CONFIG_MOUSE_SYNAPTICS_USB=m
+# CONFIG_NET_VENDOR_VERTEXCOM is not set
+CONFIG_PCI_MSI=y
+CONFIG_PHYLIB=y
+CONFIG_PINCTRL_AMD=y
+CONFIG_PINCTRL_BAYTRAIL=y
+CONFIG_PINCTRL_BROXTON=y
+CONFIG_PINCTRL_CANNONLAKE=y
+CONFIG_PINCTRL_CHERRYVIEW=y
+CONFIG_PINCTRL_GEMINILAKE=y
+CONFIG_PINCTRL_ICELAKE=y
+CONFIG_PINCTRL_SUNRISEPOINT=y
+CONFIG_PINCTRL_TIGERLAKE=y
+CONFIG_PROCESSOR_SELECT=y
+CONFIG_PTP_1588_CLOCK=m
+CONFIG_RTL8187=m
+CONFIG_RTL8192CU=m
+# CONFIG_RTLWIFI_DEBUG is not set
+CONFIG_RTW88=m
+CONFIG_RTW88_8822BE=m
+CONFIG_RTW88_8822CE=m
+CONFIG_RTW88_DEBUG=y
+CONFIG_RTW88_DEBUGFS=y
+CONFIG_SATA_AHCI=y
+CONFIG_SATA_MOBILE_LPM_POLICY=3
+# CONFIG_SATA_PMP is not set
+CONFIG_SCHED_CORE=y
+CONFIG_SCSI_VIRTIO=y
+CONFIG_SKGE=m
+CONFIG_SKY2=m
+CONFIG_SND_HDA_CODEC_ANALOG=m
+CONFIG_SND_HDA_CODEC_CA0110=m
+CONFIG_SND_HDA_CODEC_CA0132=m
+CONFIG_SND_HDA_CODEC_CIRRUS=m
+CONFIG_SND_HDA_CODEC_CMEDIA=m
+CONFIG_SND_HDA_CODEC_CONEXANT=m
+CONFIG_SND_HDA_CODEC_REALTEK=m
+CONFIG_SND_HDA_CODEC_SI3054=m
+CONFIG_SND_HDA_CODEC_SIGMATEL=m
+CONFIG_SND_HDA_CODEC_VIA=m
+CONFIG_SND_HDA_POWER_SAVE_DEFAULT=15
+CONFIG_SND_SOC_ADAU7002=m
+CONFIG_SND_SOC_INTEL_BXT_DA7219_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_CML_H=m
+CONFIG_SND_SOC_INTEL_CML_LP=m
+CONFIG_SND_SOC_INTEL_CML_LP_DA7219_MAX98357A_MACH=m
+CONFIG_SND_SOC_INTEL_SKYLAKE=m
+CONFIG_SND_SOC_MAX98927=m
+CONFIG_SND_SOC_SOF_ACPI=m
+CONFIG_SND_SOC_SOF_HDA_LINK=y
+CONFIG_SND_SOC_SOF_INTEL_TOPLEVEL=y
+CONFIG_SND_SOC_SOF_PCI=m
+CONFIG_SND_SOC_SOF_TOPLEVEL=y
+CONFIG_SND_SOC_SSM4567=m
+CONFIG_SND_SOC_TS3A227E=y
+CONFIG_SOUNDWIRE=m
+CONFIG_SX9310=m
+CONFIG_TOUCHSCREEN_RM_TS=y
+CONFIG_VIRTIO_BLK=m
+CONFIG_VIRTIO_NET=m
+CONFIG_VIRTIO_PCI=y
+CONFIG_X86_INTEL_LPSS=y
+# CONFIG_X86_MCE_AMD is not set
+CONFIG_X86_PKG_TEMP_THERMAL=y
+CONFIG_ZRAM=m
+CONFIG_ZRAM_MEMORY_TRACKING=y
+CONFIG_ZRAM_WRITEBACK=y
diff --git a/chromeos/config/chromeos/x86_64/common.config b/chromeos/config/chromeos/x86_64/common.config
new file mode 100644
index 000000000000..189598bf80ae
--- /dev/null
+++ b/chromeos/config/chromeos/x86_64/common.config
@@ -0,0 +1,145 @@
+#
+# Config options generated by splitconfig
+#
+CONFIG_ACPI_ALS=m
+CONFIG_ACPI_APEI=y
+CONFIG_ACPI_CHROMEOS=y
+CONFIG_ACPI_DOCK=y
+CONFIG_ALT_SYSCALL=y
+CONFIG_ALT_SYSCALL_CHROMIUMOS=y
+CONFIG_ALX=m
+CONFIG_AQTION=m
+CONFIG_ARCH_MMAP_RND_BITS=31
+CONFIG_ARCH_MMAP_RND_COMPAT_BITS=16
+CONFIG_ATH9K=m
+CONFIG_ATH9K_DEBUGFS=y
+# CONFIG_ATH9K_RFKILL is not set
+CONFIG_ATH_DEBUG=y
+CONFIG_BLK_DEV_NVME=y
+CONFIG_BRCMFMAC=m
+CONFIG_BRCMFMAC_PCIE=y
+CONFIG_BRCMFMAC_USB=y
+CONFIG_BT_ATH3K=m
+CONFIG_BT_HCIBCM203X=m
+CONFIG_CHARGER_CROS_USBPD=y
+CONFIG_CHROMEOS_LAPTOP=y
+CONFIG_CHROMEOS_PSTORE=y
+CONFIG_CHROMEOS_TBMC=y
+CONFIG_CONFIGFS_FS=y
+CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
+CONFIG_CROS_EC_I2C=y
+CONFIG_CROS_EC_LPC=y
+CONFIG_CROS_KBD_LED_BACKLIGHT=y
+CONFIG_CRYPTO_AES_NI_INTEL=y
+CONFIG_CRYPTO_LRW=y
+CONFIG_CRYPTO_SHA256_SSSE3=y
+CONFIG_DEBUG_BOOT_PARAMS=y
+CONFIG_DMI_SYSFS=y
+CONFIG_DRM_DP_CEC=y
+CONFIG_E100=m
+CONFIG_E1000=m
+CONFIG_E1000E=m
+CONFIG_EFI=y
+# CONFIG_EFI_VARS_PSTORE is not set
+CONFIG_FTRACE_SYSCALLS=y
+CONFIG_GART_IOMMU=y
+CONFIG_GOOGLE_SMI=y
+CONFIG_HARDLOCKUP_DETECTOR=y
+CONFIG_HID_GOOGLE_HAMMER=m
+CONFIG_HID_PID=y
+CONFIG_HID_RMI=m
+CONFIG_HOTPLUG_PCI=y
+CONFIG_HOTPLUG_PCI_PCIE=y
+CONFIG_HPET=y
+# CONFIG_HPET_MMAP is not set
+# CONFIG_HW_RANDOM_VIA is not set
+CONFIG_HZ_1000=y
+CONFIG_I2C_CHARDEV=y
+CONFIG_I2C_DESIGNWARE_PCI=y
+CONFIG_I2C_DESIGNWARE_PLATFORM=y
+CONFIG_I2C_HID_ACPI=y
+CONFIG_I2C_I801=y
+CONFIG_I2C_PIIX4=m
+CONFIG_IA32_EMULATION=y
+CONFIG_IGBVF=m
+CONFIG_IIO=m
+CONFIG_IIO_CROS_EC_ACCEL_LEGACY=m
+CONFIG_IIO_CROS_EC_ACTIVITY=m
+CONFIG_IIO_CROS_EC_BARO=m
+CONFIG_IIO_CROS_EC_LIGHT_PROX=m
+CONFIG_IIO_CROS_EC_SENSORS=m
+CONFIG_IIO_CROS_EC_SENSORS_LID_ANGLE=m
+CONFIG_IIO_CROS_EC_SENSORS_SYNC=m
+CONFIG_INTEL_RAPL=y
+CONFIG_INTEL_VBTN=m
+CONFIG_IO_DELAY_0XED=y
+CONFIG_JME=m
+CONFIG_KFENCE=y
+CONFIG_KFENCE_SAMPLE_INTERVAL=500
+CONFIG_LPC_ICH=y
+CONFIG_LPC_SCH=y
+CONFIG_LSM_MMAP_MIN_ADDR=32768
+CONFIG_MEDIA_CEC_SUPPORT=y
+CONFIG_MEDIA_PCI_SUPPORT=y
+# CONFIG_MMC_RICOH_MMC is not set
+CONFIG_MMC_SDHCI_ACPI=y
+CONFIG_MMC_SDHCI_PCI=y
+CONFIG_MOUSE_APPLETOUCH=m
+# CONFIG_MOUSE_PS2_ALPS is not set
+# CONFIG_MOUSE_PS2_CYPRESS is not set
+# CONFIG_MOUSE_PS2_FOCALTECH is not set
+# CONFIG_MOUSE_PS2_LIFEBOOK is not set
+# CONFIG_MOUSE_PS2_LOGIPS2PP is not set
+# CONFIG_MOUSE_PS2_SYNAPTICS is not set
+# CONFIG_MTRR_SANITIZER is not set
+CONFIG_MWIFIEX_PCIE=m
+CONFIG_PCI=y
+CONFIG_PCIEAER=y
+CONFIG_PCIEPORTBUS=y
+# CONFIG_PCSPKR_PLATFORM is not set
+CONFIG_PM_TRACE_RTC=y
+CONFIG_POWERCAP=y
+CONFIG_PROC_KCORE=y
+CONFIG_R8169=m
+CONFIG_RT2800PCI=m
+# CONFIG_RTC_HCTOSYS is not set
+CONFIG_RTW89=m
+CONFIG_RTW89_8852AE=m
+CONFIG_RTW89_DEBUGFS=y
+CONFIG_RTW89_DEBUGMSG=y
+CONFIG_SCSI_CONSTANTS=y
+CONFIG_SECURITY_CHROMIUMOS=y
+CONFIG_SENSORS_CORETEMP=y
+# CONFIG_SERIO_SERPORT is not set
+CONFIG_SMP=y
+CONFIG_SND_HDA_CODEC_HDMI=m
+CONFIG_SND_HDA_HWDEP=y
+CONFIG_SND_HDA_INTEL=m
+CONFIG_SND_HDA_PATCH_LOADER=y
+CONFIG_SPI_BITBANG=m
+CONFIG_SPI_PXA2XX=y
+CONFIG_TCG_TIS=y
+CONFIG_TCG_TIS_SPI=y
+CONFIG_TCG_TIS_SPI_CR50=y
+CONFIG_TOUCHSCREEN_MELFAS_MIP4=y
+CONFIG_TOUCHSCREEN_WDT87XX_I2C=y
+CONFIG_TYPEC=m
+CONFIG_TYPEC_UCSI=m
+CONFIG_UCSI_ACPI=m
+CONFIG_USB_DWC3=m
+CONFIG_USB_DWC3_GADGET=y
+CONFIG_USB_EHCI_ROOT_HUB_TT=y
+CONFIG_USB_GADGET=m
+CONFIG_USB_STORAGE_REALTEK=y
+CONFIG_USB_UHCI_HCD=y
+CONFIG_USB_XHCI_HCD=y
+# CONFIG_VGA_ARB is not set
+# CONFIG_X86_5LEVEL is not set
+CONFIG_X86_ACPI_CPUFREQ=y
+CONFIG_X86_AMD_PLATFORM_DEVICE=y
+CONFIG_X86_CHECK_BIOS_CORRUPTION=y
+CONFIG_X86_CPUID=y
+CONFIG_X86_MSR=y
+CONFIG_X86_REROUTE_FOR_BROKEN_BOOT_IRQS=y
+# CONFIG_XZ_DEC_ARM is not set
+# CONFIG_XZ_DEC_ARMTHUMB is not set
-- 
2.39.0.314.g84b9a713c41-goog

